<div class="mw-parser-output"><table class="plainlinks metadata ambox ambox-content ambox-Refimprove" role="presentation">
<tr>
<td class="mbox-image">
<div style="width:52px"><a href="/wiki/File:Question_book-new.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/en/thumb/9/99/Question_book-new.svg/50px-Question_book-new.svg.png" width="50" height="39" srcset="//upload.wikimedia.org/wikipedia/en/thumb/9/99/Question_book-new.svg/75px-Question_book-new.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/9/99/Question_book-new.svg/100px-Question_book-new.svg.png 2x" data-file-width="512" data-file-height="399" /></a></div>
</td>
<td class="mbox-text"><span class="mbox-text-span">This article <b>needs additional citations for <a href="/wiki/Wikipedia:Verifiability" title="Wikipedia:Verifiability">verification</a></b>. <span class="hide-when-compact">Please help <a class="external text" href="//en.wikipedia.org/w/index.php?title=Sequence_alignment&amp;action=edit">improve this article</a> by <a href="/wiki/Help:Introduction_to_referencing_with_Wiki_Markup/1" title="Help:Introduction to referencing with Wiki Markup/1">adding citations to reliable sources</a>. Unsourced material may be challenged and removed.</span> <small><i>(March 2009)</i></small> <small class="hide-when-compact"><i>(<a href="/wiki/Help:Maintenance_template_removal" title="Help:Maintenance template removal">Learn how and when to remove this template message</a>)</i></small></span></td>
</tr>
</table>
<p>In <a href="/wiki/Bioinformatics" title="Bioinformatics">bioinformatics</a>, a <b>sequence alignment</b> is a way of arranging the sequences of <a href="/wiki/DNA" title="DNA">DNA</a>, <a href="/wiki/RNA" title="RNA">RNA</a>, or <a href="/wiki/Protein" title="Protein">protein</a> to identify regions of similarity that may be a consequence of functional, <a href="/wiki/Structural_biology" title="Structural biology">structural</a>, or <a href="/wiki/Evolution" title="Evolution">evolutionary</a> relationships between the sequences.<sup id="cite_ref-mount_1-0" class="reference"><a href="#cite_note-mount-1">[1]</a></sup> Aligned sequences of <a href="/wiki/Nucleotide" title="Nucleotide">nucleotide</a> or <a href="/wiki/Amino_acid" title="Amino acid">amino acid</a> residues are typically represented as rows within a <a href="/wiki/Matrix_(mathematics)" title="Matrix (mathematics)">matrix</a>. Gaps are inserted between the <a href="/wiki/Residue_(chemistry)" title="Residue (chemistry)">residues</a> so that identical or similar characters are aligned in successive columns.</p>
<div class="center">
<div class="thumb tnone">
<div class="thumbinner" style="width:721px;"><a href="/wiki/File:Zinc-finger-seq-alignment2.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/8/86/Zinc-finger-seq-alignment2.png" width="719" height="135" class="thumbimage" data-file-width="719" data-file-height="135" /></a>
<div class="thumbcaption">A sequence alignment, produced by <a href="/wiki/ClustalW" class="mw-redirect" title="ClustalW">ClustalW</a>, of two <a href="/wiki/Human" title="Human">human</a> <a href="/wiki/Zinc_finger" title="Zinc finger">zinc finger</a> proteins, identified on the left by <a href="/wiki/GenBank" title="GenBank">GenBank</a> accession number.<br />
Key: Single letters: <a href="/wiki/Amino_acid#Table_of_standard_amino_acid_abbreviations_and_side_chain_properties" title="Amino acid">amino acids</a>. Red: small, hydrophobic, aromatic, not Y. Blue: acidic. Magenta: basic. Green: hydroxyl, amine, amide, basic. Gray: others. "*": identical. ":": conserved substitutions (same colour group). ".": semi-conserved substitution (similar shapes).<sup id="cite_ref-2" class="reference"><a href="#cite_note-2">[2]</a></sup></div>
</div>
</div>
</div>
<p>Sequence alignments are also used for non-biological sequences, such as those present in <a href="/wiki/Natural_language" title="Natural language">natural language</a> or in financial data.</p>
<p></p>
<div id="toc" class="toc">
<div class="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#Interpretation"><span class="tocnumber">1</span> <span class="toctext">Interpretation</span></a></li>
<li class="toclevel-1 tocsection-2"><a href="#Alignment_methods"><span class="tocnumber">2</span> <span class="toctext">Alignment methods</span></a></li>
<li class="toclevel-1 tocsection-3"><a href="#Representations"><span class="tocnumber">3</span> <span class="toctext">Representations</span></a></li>
<li class="toclevel-1 tocsection-4"><a href="#Global_and_local_alignments"><span class="tocnumber">4</span> <span class="toctext">Global and local alignments</span></a></li>
<li class="toclevel-1 tocsection-5"><a href="#Pairwise_alignment"><span class="tocnumber">5</span> <span class="toctext">Pairwise alignment</span></a>
<ul>
<li class="toclevel-2 tocsection-6"><a href="#Dot-matrix_methods"><span class="tocnumber">5.1</span> <span class="toctext">Dot-matrix methods</span></a></li>
<li class="toclevel-2 tocsection-7"><a href="#Dynamic_programming"><span class="tocnumber">5.2</span> <span class="toctext">Dynamic programming</span></a></li>
<li class="toclevel-2 tocsection-8"><a href="#Word_methods"><span class="tocnumber">5.3</span> <span class="toctext">Word methods</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-9"><a href="#Multiple_sequence_alignment"><span class="tocnumber">6</span> <span class="toctext">Multiple sequence alignment</span></a>
<ul>
<li class="toclevel-2 tocsection-10"><a href="#Dynamic_programming_2"><span class="tocnumber">6.1</span> <span class="toctext">Dynamic programming</span></a></li>
<li class="toclevel-2 tocsection-11"><a href="#Progressive_methods"><span class="tocnumber">6.2</span> <span class="toctext">Progressive methods</span></a></li>
<li class="toclevel-2 tocsection-12"><a href="#Iterative_methods"><span class="tocnumber">6.3</span> <span class="toctext">Iterative methods</span></a></li>
<li class="toclevel-2 tocsection-13"><a href="#Motif_finding"><span class="tocnumber">6.4</span> <span class="toctext">Motif finding</span></a></li>
<li class="toclevel-2 tocsection-14"><a href="#Techniques_inspired_by_computer_science"><span class="tocnumber">6.5</span> <span class="toctext">Techniques inspired by computer science</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-15"><a href="#Structural_alignment"><span class="tocnumber">7</span> <span class="toctext">Structural alignment</span></a>
<ul>
<li class="toclevel-2 tocsection-16"><a href="#DALI"><span class="tocnumber">7.1</span> <span class="toctext">DALI</span></a></li>
<li class="toclevel-2 tocsection-17"><a href="#SSAP"><span class="tocnumber">7.2</span> <span class="toctext">SSAP</span></a></li>
<li class="toclevel-2 tocsection-18"><a href="#Combinatorial_extension"><span class="tocnumber">7.3</span> <span class="toctext">Combinatorial extension</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-19"><a href="#Phylogenetic_analysis"><span class="tocnumber">8</span> <span class="toctext">Phylogenetic analysis</span></a>
<ul>
<li class="toclevel-2 tocsection-20"><a href="#Assessment_of_significance"><span class="tocnumber">8.1</span> <span class="toctext">Assessment of significance</span></a></li>
<li class="toclevel-2 tocsection-21"><a href="#Assessment_of_credibility"><span class="tocnumber">8.2</span> <span class="toctext">Assessment of credibility</span></a></li>
<li class="toclevel-2 tocsection-22"><a href="#Scoring_functions"><span class="tocnumber">8.3</span> <span class="toctext">Scoring functions</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-23"><a href="#Other_biological_uses"><span class="tocnumber">9</span> <span class="toctext">Other biological uses</span></a></li>
<li class="toclevel-1 tocsection-24"><a href="#Non-biological_uses"><span class="tocnumber">10</span> <span class="toctext">Non-biological uses</span></a></li>
<li class="toclevel-1 tocsection-25"><a href="#Software"><span class="tocnumber">11</span> <span class="toctext">Software</span></a></li>
<li class="toclevel-1 tocsection-26"><a href="#See_also"><span class="tocnumber">12</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-27"><a href="#References"><span class="tocnumber">13</span> <span class="toctext">References</span></a></li>
</ul>
</div>
<p></p>
<h2><span class="mw-headline" id="Interpretation">Interpretation</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=1" title="Edit section: Interpretation">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>If two sequences in an alignment share a common ancestor, mismatches can be interpreted as <a href="/wiki/Point_mutation" title="Point mutation">point mutations</a> and gaps as <a href="/wiki/Indel" title="Indel">indels</a> (that is, insertion or deletion mutations) introduced in one or both lineages in the time since they diverged from one another. In sequence alignments of proteins, the degree of similarity between <a href="/wiki/Amino_acid" title="Amino acid">amino acids</a> occupying a particular position in the sequence can be interpreted as a rough measure of how <a href="/wiki/Conservation_(genetics)" class="mw-redirect" title="Conservation (genetics)">conserved</a> a particular region or <a href="/wiki/Sequence_motif" title="Sequence motif">sequence motif</a> is among lineages. The absence of substitutions, or the presence of only very conservative substitutions (that is, the substitution of amino acids whose <a href="/wiki/Side_chain" title="Side chain">side chains</a> have similar biochemical properties) in a particular region of the sequence, suggest <sup id="cite_ref-predict_3-0" class="reference"><a href="#cite_note-predict-3">[3]</a></sup> that this region has structural or functional importance. Although DNA and RNA <a href="/wiki/Nucleotide" title="Nucleotide">nucleotide</a> bases are more similar to each other than are amino acids, the conservation of base pairs can indicate a similar functional or structural role.</p>
<h2><span class="mw-headline" id="Alignment_methods">Alignment methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=2" title="Edit section: Alignment methods">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Very short or very similar sequences can be aligned by hand. However, most interesting problems require the alignment of lengthy, highly variable or extremely numerous sequences that cannot be aligned solely by human effort. Instead, human knowledge is applied in constructing algorithms to produce high-quality sequence alignments, and occasionally in adjusting the final results to reflect patterns that are difficult to represent algorithmically (especially in the case of nucleotide sequences). Computational approaches to sequence alignment generally fall into two categories: <i>global alignments</i> and <i>local alignments</i>. Calculating a global alignment is a form of <a href="/wiki/Global_optimization" title="Global optimization">global optimization</a> that "forces" the alignment to span the entire length of all query sequences. By contrast, local alignments identify regions of similarity within long sequences that are often widely divergent overall. Local alignments are often preferable, but can be more difficult to calculate because of the additional challenge of identifying the regions of similarity. A variety of computational algorithms have been applied to the sequence alignment problem, including slow but formally correct methods like <a href="/wiki/Dynamic_programming" title="Dynamic programming">dynamic programming</a>, and efficient, <a href="/wiki/Heuristic_algorithm" class="mw-redirect" title="Heuristic algorithm">heuristic algorithms</a> or <a href="/wiki/Probability" title="Probability">probabilistic</a> methods that do not guarantee to find best matches designed for large-scale database search.</p>
<h2><span class="mw-headline" id="Representations">Representations</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=3" title="Edit section: Representations">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Alignments are commonly represented both graphically and in text format. In almost all sequence alignment representations, sequences are written in rows arranged so that aligned residues appear in successive columns. In text formats, aligned columns containing identical or similar characters are indicated with a system of conservation symbols. As in the image above, an asterisk or pipe symbol is used to show identity between two columns; other less common symbols include a colon for conservative substitutions and a period for semiconservative substitutions. Many sequence visualization programs also use color to display information about the properties of the individual sequence elements; in DNA and RNA sequences, this equates to assigning each nucleotide its own color. In protein alignments, such as the one in the image above, color is often used to indicate amino acid properties to aid in judging the <a href="/wiki/Conservation_(genetics)" class="mw-redirect" title="Conservation (genetics)">conservation</a> of a given amino acid substitution. For multiple sequences the last row in each column is often the <a href="/wiki/Consensus_sequence" title="Consensus sequence">consensus sequence</a> determined by the alignment; the consensus sequence is also often represented in graphical format with a <a href="/wiki/Sequence_logo" title="Sequence logo">sequence logo</a> in which the size of each nucleotide or amino acid letter corresponds to its degree of conservation.<sup id="cite_ref-Schneider_4-0" class="reference"><a href="#cite_note-Schneider-4">[4]</a></sup></p>
<p>Sequence alignments can be stored in a wide variety of text-based file formats, many of which were originally developed in conjunction with a specific alignment program or implementation. Most web-based tools allow a limited number of input and output formats, such as <a href="/wiki/FASTA_format" title="FASTA format">FASTA format</a> and <a href="/wiki/GenBank" title="GenBank">GenBank</a> format and the output is not easily editable. Several conversion programs that provide graphical and/or command line interfaces are available<sup class="noprint Inline-Template"><span style="white-space: nowrap;">[<i><a href="/wiki/Wikipedia:Link_rot" title="Wikipedia:Link rot"><span title="&#160;Dead link since August 2009">dead link</span></a></i>]</span></sup>, such as <a rel="nofollow" class="external text" href="http://bioweb.pasteur.fr/seqanal/interfaces/readseq.html">READSEQ</a> and <a href="/wiki/EMBOSS" title="EMBOSS">EMBOSS</a>. There are also several programming packages which provide this conversion functionality, such as <a href="/wiki/BioPerl" title="BioPerl">BioPerl</a> and <a href="/wiki/BioRuby" title="BioRuby">BioRuby</a>.</p>
<h2><span class="mw-headline" id="Global_and_local_alignments">Global and local alignments</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=4" title="Edit section: Global and local alignments">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright">
<div class="thumbinner" style="width:245px;"><a href="/wiki/File:Global-local-alignment.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/4/4b/Global-local-alignment.png" width="243" height="109" class="thumbimage" data-file-width="243" data-file-height="109" /></a>
<div class="thumbcaption">Illustration of global and local alignments demonstrating the 'gappy' quality of global alignments that can occur if sequences are insufficiently similar</div>
</div>
</div>
<p>Global alignments, which attempt to align every residue in every sequence, are most useful when the sequences in the query set are similar and of roughly equal size. (This does not mean global alignments cannot end in gaps.) A general global alignment technique is the <a href="/wiki/Needleman%E2%80%93Wunsch_algorithm" title="Needleman–Wunsch algorithm">Needleman–Wunsch algorithm</a>, which is based on dynamic programming. Local alignments are more useful for dissimilar sequences that are suspected to contain regions of similarity or similar sequence motifs within their larger sequence context. The <a href="/wiki/Smith%E2%80%93Waterman_algorithm" title="Smith–Waterman algorithm">Smith–Waterman algorithm</a> is a general local alignment method also based on dynamic programming. With sufficiently similar sequences, there is no difference between local and global alignments.</p>
<p>Hybrid methods, known as semiglobal or "glocal" (short for <b>glo</b>bal-lo<b>cal</b>) methods, attempt to find the best possible alignment that includes the start and end of one or the other sequence. This can be especially useful when the downstream part of one sequence overlaps with the upstream part of the other sequence. In this case, neither global nor local alignment is entirely appropriate: a global alignment would attempt to force the alignment to extend beyond the region of overlap, while a local alignment might not fully cover the region of overlap.<sup id="cite_ref-brudno_5-0" class="reference"><a href="#cite_note-brudno-5">[5]</a></sup></p>
<h2><span class="mw-headline" id="Pairwise_alignment">Pairwise alignment</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=5" title="Edit section: Pairwise alignment">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Pairwise sequence alignment methods are used to find the best-matching piecewise (local) or global alignments of two query sequences. Pairwise alignments can only be used between two sequences at a time, but they are efficient to calculate and are often used for methods that do not require extreme precision (such as searching a database for sequences with high similarity to a query). The three primary methods of producing pairwise alignments are dot-matrix methods, dynamic programming, and word methods;<sup id="cite_ref-mount_1-1" class="reference"><a href="#cite_note-mount-1">[1]</a></sup> however, multiple sequence alignment techniques can also align pairs of sequences. Although each method has its individual strengths and weaknesses, all three pairwise methods have difficulty with highly repetitive sequences of low <a href="/wiki/Information_content" class="mw-redirect" title="Information content">information content</a> - especially where the number of repetitions differ in the two sequences to be aligned. One way of quantifying the utility of a given pairwise alignment is the 'maximum unique match' (MUM), or the longest subsequence that occurs in both query sequence. Longer MUM sequences typically reflect closer relatedness.</p>
<h3><span class="mw-headline" id="Dot-matrix_methods">Dot-matrix methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=6" title="Edit section: Dot-matrix methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<table role="presentation" class="mbox-small plainlinks sistersitebox" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
<tr>
<td class="mbox-image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/91/Wikiversity-logo.svg/40px-Wikiversity-logo.svg.png" width="40" height="32" class="noviewer" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/91/Wikiversity-logo.svg/60px-Wikiversity-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/91/Wikiversity-logo.svg/80px-Wikiversity-logo.svg.png 2x" data-file-width="1000" data-file-height="800" /></td>
<td class="mbox-text plainlist">Wikiversity has learning resources about <i><b><a href="https://en.wikiversity.org/wiki/Dot-matrix_methods" class="extiw" title="v:Dot-matrix methods">Dot-matrix methods</a></b></i></td>
</tr>
</table>
<div class="thumb tright">
<div class="thumbinner" style="width:202px;"><a href="/wiki/File:Zinc-finger-dot-plot.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/3/33/Zinc-finger-dot-plot.png/200px-Zinc-finger-dot-plot.png" width="200" height="200" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/3/33/Zinc-finger-dot-plot.png/300px-Zinc-finger-dot-plot.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/3/33/Zinc-finger-dot-plot.png/400px-Zinc-finger-dot-plot.png 2x" data-file-width="404" data-file-height="404" /></a>
<div class="thumbcaption">
<div class="magnify"><a href="/wiki/File:Zinc-finger-dot-plot.png" class="internal" title="Enlarge"></a></div>
A DNA <a href="/wiki/Dot_plot_(bioinformatics)" title="Dot plot (bioinformatics)">dot plot</a> of a <a href="/wiki/Human" title="Human">human</a> <a href="/wiki/Zinc_finger" title="Zinc finger">zinc finger</a> <a href="/wiki/Transcription_factor" title="Transcription factor">transcription factor</a> (GenBank ID NM_002383), showing regional <a href="/wiki/Self-similarity" title="Self-similarity">self-similarity</a>. The main diagonal represents the sequence's alignment with itself; lines off the main diagonal represent similar or repetitive patterns within the sequence. This is a typical example of a <a href="/wiki/Recurrence_plot" title="Recurrence plot">recurrence plot</a>.</div>
</div>
</div>
<p>The dot-matrix approach, which implicitly produces a family of alignments for individual sequence regions, is qualitative and conceptually simple, though time-consuming to analyze on a large scale. In the absence of noise, it can be easy to visually identify certain sequence features—such as insertions, deletions, repeats, or <a href="/wiki/Inverted_repeat" title="Inverted repeat">inverted repeats</a>—from a dot-matrix plot. To construct a dot-matrix plot, the two sequences are written along the top row and leftmost column of a two-dimensional <a href="/wiki/Matrix_(mathematics)" title="Matrix (mathematics)">matrix</a> and a dot is placed at any point where the characters in the appropriate columns match—this is a typical <a href="/wiki/Recurrence_plot" title="Recurrence plot">recurrence plot</a>. Some implementations vary the size or intensity of the dot depending on the degree of similarity of the two characters, to accommodate conservative substitutions. The dot plots of very closely related sequences will appear as a single line along the matrix's <a href="/wiki/Main_diagonal" title="Main diagonal">main diagonal</a>.</p>
<p>Problems with dot plots as an information display technique include: noise, lack of clarity, non-intuitiveness, difficulty extracting match summary statistics and match positions on the two sequences. There is also much wasted space where the match data is inherently duplicated across the diagonal and most of the actual area of the plot is taken up by either empty space or noise, and, finally, dot-plots are limited to two sequences. None of these limitations apply to Miropeats alignment diagrams but they have their own particular flaws.</p>
<p>Dot plots can also be used to assess repetitiveness in a single sequence. A sequence can be plotted against itself and regions that share significant similarities will appear as lines off the main diagonal. This effect can occur when a protein consists of multiple similar <a href="/wiki/Structural_domain" class="mw-redirect" title="Structural domain">structural domains</a>.</p>
<h3><span class="mw-headline" id="Dynamic_programming">Dynamic programming</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=7" title="Edit section: Dynamic programming">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The technique of <a href="/wiki/Dynamic_programming" title="Dynamic programming">dynamic programming</a> can be applied to produce global alignments via the <a href="/wiki/Needleman-Wunsch_algorithm" class="mw-redirect" title="Needleman-Wunsch algorithm">Needleman-Wunsch algorithm</a>, and local alignments via the <a href="/wiki/Smith-Waterman_algorithm" class="mw-redirect" title="Smith-Waterman algorithm">Smith-Waterman algorithm</a>. In typical usage, protein alignments use a <a href="/wiki/Substitution_matrix" title="Substitution matrix">substitution matrix</a> to assign scores to amino-acid matches or mismatches, and a <a href="/wiki/Gap_penalty" title="Gap penalty">gap penalty</a> for matching an amino acid in one sequence to a gap in the other. DNA and RNA alignments may use a scoring matrix, but in practice often simply assign a positive match score, a negative mismatch score, and a negative gap penalty. (In standard dynamic programming, the score of each amino acid position is independent of the identity of its neighbors, and therefore <a href="/wiki/Base_stacking" class="mw-redirect" title="Base stacking">base stacking</a> effects are not taken into account. However, it is possible to account for such effects by modifying the algorithm.) A common extension to standard linear gap costs, is the usage of two different gap penalties for opening a gap and for extending a gap. Typically the former is much larger than the latter, e.g. -10 for gap open and -2 for gap extension. Thus, the number of gaps in an alignment is usually reduced and residues and gaps are kept together, which typically makes more biological sense. The Gotoh algorithm implements affine gap costs by using three matrices.</p>
<p>Dynamic programming can be useful in aligning nucleotide to protein sequences, a task complicated by the need to take into account <a href="/wiki/Frameshift" class="mw-redirect" title="Frameshift">frameshift</a> mutations (usually insertions or deletions). The framesearch method produces a series of global or local pairwise alignments between a query nucleotide sequence and a search set of protein sequences, or vice versa. Its ability to evaluate frameshifts offset by an arbitrary number of nucleotides makes the method useful for sequences containing large numbers of indels, which can be very difficult to align with more efficient heuristic methods. In practice, the method requires large amounts of computing power or a system whose architecture is specialized for dynamic programming. The <a href="/wiki/BLAST" title="BLAST">BLAST</a> and <a href="/wiki/EMBOSS" title="EMBOSS">EMBOSS</a> suites provide basic tools for creating translated alignments (though some of these approaches take advantage of side-effects of sequence searching capabilities of the tools). More general methods are available from both commercial sources, such as <i>FrameSearch</i>, distributed as part of the <a href="/wiki/Accelrys" title="Accelrys">Accelrys</a> <a href="/wiki/GCG_(software)" class="mw-redirect" title="GCG (software)">GCG package</a>, and <a href="/wiki/Open_Source" class="mw-redirect" title="Open Source">Open Source</a> software such as <a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/Wise2">Genewise</a>.</p>
<p>The dynamic programming method is guaranteed to find an optimal alignment given a particular scoring function; however, identifying a good scoring function is often an empirical rather than a theoretical matter. Although dynamic programming is extensible to more than two sequences, it is prohibitively slow for large numbers of or extremely long sequences.</p>
<h3><span class="mw-headline" id="Word_methods">Word methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=8" title="Edit section: Word methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Word methods, also known as <i>k</i>-tuple methods, are <a href="/wiki/Heuristic" title="Heuristic">heuristic</a> methods that are not guaranteed to find an optimal alignment solution, but are significantly more efficient than dynamic programming. These methods are especially useful in large-scale database searches where it is understood that a large proportion of the candidate sequences will have essentially no significant match with the query sequence. Word methods are best known for their implementation in the database search tools <a href="/wiki/FASTA" title="FASTA">FASTA</a> and the <a href="/wiki/BLAST" title="BLAST">BLAST</a> family.<sup id="cite_ref-mount_1-2" class="reference"><a href="#cite_note-mount-1">[1]</a></sup> Word methods identify a series of short, nonoverlapping subsequences ("words") in the query sequence that are then matched to candidate database sequences. The relative positions of the word in the two sequences being compared are subtracted to obtain an offset; this will indicate a region of alignment if multiple distinct words produce the same offset. Only if this region is detected do these methods apply more sensitive alignment criteria; thus, many unnecessary comparisons with sequences of no appreciable similarity are eliminated.</p>
<p>In the FASTA method, the user defines a value <i>k</i> to use as the word length with which to search the database. The method is slower but more sensitive at lower values of <i>k</i>, which are also preferred for searches involving a very short query sequence. The BLAST family of search methods provides a number of algorithms optimized for particular types of queries, such as searching for distantly related sequence matches. BLAST was developed to provide a faster alternative to FASTA without sacrificing much accuracy; like FASTA, BLAST uses a word search of length <i>k</i>, but evaluates only the most significant word matches, rather than every word match as does FASTA. Most BLAST implementations use a fixed default word length that is optimized for the query and database type, and that is changed only under special circumstances, such as when searching with repetitive or very short query sequences. Implementations can be found via a number of web portals, such as <a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/fasta33/">EMBL FASTA</a> and <a rel="nofollow" class="external text" href="http://www.ncbi.nlm.nih.gov/BLAST/">NCBI BLAST</a>.</p>
<h2><span class="mw-headline" id="Multiple_sequence_alignment">Multiple sequence alignment</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=9" title="Edit section: Multiple sequence alignment">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Multiple_sequence_alignment" title="Multiple sequence alignment">Multiple sequence alignment</a></div>
<div class="thumb tright">
<div class="thumbinner" style="width:302px;"><a href="/wiki/File:Hemagglutinin-alignments.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Hemagglutinin-alignments.png/300px-Hemagglutinin-alignments.png" width="300" height="322" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Hemagglutinin-alignments.png/450px-Hemagglutinin-alignments.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Hemagglutinin-alignments.png/600px-Hemagglutinin-alignments.png 2x" data-file-width="954" data-file-height="1025" /></a>
<div class="thumbcaption">
<div class="magnify"><a href="/wiki/File:Hemagglutinin-alignments.png" class="internal" title="Enlarge"></a></div>
Alignment of 27 <a href="/wiki/Avian_influenza" title="Avian influenza">avian influenza</a> <a href="/wiki/Hemagglutinin" title="Hemagglutinin">hemagglutinin</a> protein sequences colored by residue conservation (top) and residue properties (bottom)</div>
</div>
</div>
<p><a href="/wiki/Multiple_sequence_alignment" title="Multiple sequence alignment">Multiple sequence alignment</a> is an extension of pairwise alignment to incorporate more than two sequences at a time. Multiple alignment methods try to align all of the sequences in a given query set. Multiple alignments are often used in identifying <a href="/wiki/Conservation_(genetics)" class="mw-redirect" title="Conservation (genetics)">conserved</a> sequence regions across a group of sequences hypothesized to be evolutionarily related. Such conserved sequence motifs can be used in conjunction with structural and <a href="/wiki/Reaction_mechanism" title="Reaction mechanism">mechanistic</a> information to locate the catalytic <a href="/wiki/Active_site" title="Active site">active sites</a> of <a href="/wiki/Enzyme" title="Enzyme">enzymes</a>. Alignments are also used to aid in establishing evolutionary relationships by constructing <a href="/wiki/Phylogenetic_tree" title="Phylogenetic tree">phylogenetic trees</a>. Multiple sequence alignments are computationally difficult to produce and most formulations of the problem lead to <a href="/wiki/NP-complete" class="mw-redirect" title="NP-complete">NP-complete</a> combinatorial optimization problems.<sup id="cite_ref-wang_6-0" class="reference"><a href="#cite_note-wang-6">[6]</a></sup><sup id="cite_ref-elias_7-0" class="reference"><a href="#cite_note-elias-7">[7]</a></sup> Nevertheless, the utility of these alignments in bioinformatics has led to the development of a variety of methods suitable for aligning three or more sequences.</p>
<h3><span class="mw-headline" id="Dynamic_programming_2">Dynamic programming</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=10" title="Edit section: Dynamic programming">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The technique of dynamic programming is theoretically applicable to any number of sequences; however, because it is computationally expensive in both time and <a href="/wiki/Computer_memory" title="Computer memory">memory</a>, it is rarely used for more than three or four sequences in its most basic form. This method requires constructing the <i>n</i>-dimensional equivalent of the sequence matrix formed from two sequences, where <i>n</i> is the number of sequences in the query. Standard dynamic programming is first used on all pairs of query sequences and then the "alignment space" is filled in by considering possible matches or gaps at intermediate positions, eventually constructing an alignment essentially between each two-sequence alignment. Although this technique is computationally expensive, its guarantee of a global optimum solution is useful in cases where only a few sequences need to be aligned accurately. One method for reducing the computational demands of dynamic programming, which relies on the "sum of pairs" <a href="/wiki/Objective_function" class="mw-redirect" title="Objective function">objective function</a>, has been implemented in the <a rel="nofollow" class="external text" href="http://www.ncbi.nlm.nih.gov/CBBresearch/Schaffer/msa.html">MSA</a> software package.<sup id="cite_ref-lipman_8-0" class="reference"><a href="#cite_note-lipman-8">[8]</a></sup></p>
<h3><span class="mw-headline" id="Progressive_methods">Progressive methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=11" title="Edit section: Progressive methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Progressive, hierarchical, or tree methods generate a multiple sequence alignment by first aligning the most similar sequences and then adding successively less related sequences or groups to the alignment until the entire query set has been incorporated into the solution. The initial tree describing the sequence relatedness is based on pairwise comparisons that may include heuristic pairwise alignment methods similar to <a href="/wiki/FASTA" title="FASTA">FASTA</a>. Progressive alignment results are dependent on the choice of "most related" sequences and thus can be sensitive to inaccuracies in the initial pairwise alignments. Most progressive multiple sequence alignment methods additionally weight the sequences in the query set according to their relatedness, which reduces the likelihood of making a poor choice of initial sequences and thus improves alignment accuracy.</p>
<p>Many variations of the <a href="/wiki/Clustal" title="Clustal">Clustal</a> progressive implementation<sup id="cite_ref-higgins_9-0" class="reference"><a href="#cite_note-higgins-9">[9]</a></sup><sup id="cite_ref-thompson_10-0" class="reference"><a href="#cite_note-thompson-10">[10]</a></sup><sup id="cite_ref-chenna_11-0" class="reference"><a href="#cite_note-chenna-11">[11]</a></sup> are used for multiple sequence alignment, phylogenetic tree construction, and as input for <a href="/wiki/Protein_structure_prediction" title="Protein structure prediction">protein structure prediction</a>. A slower but more accurate variant of the progressive method is known as <a href="/wiki/T-Coffee" title="T-Coffee">T-Coffee</a>.<sup id="cite_ref-notredame_12-0" class="reference"><a href="#cite_note-notredame-12">[12]</a></sup></p>
<h3><span class="mw-headline" id="Iterative_methods">Iterative methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=12" title="Edit section: Iterative methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Iterative methods attempt to improve on the heavy dependence on the accuracy of the initial pairwise alignments, which is the weak point of the progressive methods. Iterative methods optimize an <a href="/wiki/Objective_function" class="mw-redirect" title="Objective function">objective function</a> based on a selected alignment scoring method by assigning an initial global alignment and then realigning sequence subsets. The realigned subsets are then themselves aligned to produce the next iteration's multiple sequence alignment. Various ways of selecting the sequence subgroups and objective function are reviewed in.<sup id="cite_ref-hirosawa_13-0" class="reference"><a href="#cite_note-hirosawa-13">[13]</a></sup></p>
<h3><span class="mw-headline" id="Motif_finding">Motif finding</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=13" title="Edit section: Motif finding">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Motif finding, also known as profile analysis, constructs global multiple sequence alignments that attempt to align short conserved <a href="/wiki/Sequence_motif" title="Sequence motif">sequence motifs</a> among the sequences in the query set. This is usually done by first constructing a general global multiple sequence alignment, after which the highly <a href="/wiki/Conservation_(genetics)" class="mw-redirect" title="Conservation (genetics)">conserved</a> regions are isolated and used to construct a set of profile matrices. The profile matrix for each conserved region is arranged like a scoring matrix but its frequency counts for each amino acid or nucleotide at each position are derived from the conserved region's character distribution rather than from a more general empirical distribution. The profile matrices are then used to search other sequences for occurrences of the motif they characterize. In cases where the original <a href="/wiki/Data_set" title="Data set">data set</a> contained a small number of sequences, or only highly related sequences, <a href="/wiki/Pseudocount" class="mw-redirect" title="Pseudocount">pseudocounts</a> are added to normalize the character distributions represented in the motif.</p>
<h3><span class="mw-headline" id="Techniques_inspired_by_computer_science">Techniques inspired by computer science</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=14" title="Edit section: Techniques inspired by computer science">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>A variety of general <a href="/wiki/Optimization_(mathematics)" class="mw-redirect" title="Optimization (mathematics)">optimization</a> algorithms commonly used in computer science have also been applied to the multiple sequence alignment problem. <a href="/wiki/Hidden_Markov_model" title="Hidden Markov model">Hidden Markov models</a> have been used to produce probability scores for a family of possible multiple sequence alignments for a given query set; although early HMM-based methods produced underwhelming performance, later applications have found them especially effective in detecting remotely related sequences because they are less susceptible to noise created by conservative or semiconservative substitutions.<sup id="cite_ref-karplus_14-0" class="reference"><a href="#cite_note-karplus-14">[14]</a></sup> <a href="/wiki/Genetic_algorithm" title="Genetic algorithm">Genetic algorithms</a> and <a href="/wiki/Simulated_annealing" title="Simulated annealing">simulated annealing</a> have also been used in optimizing multiple sequence alignment scores as judged by a scoring function like the sum-of-pairs method. More complete details and software packages can be found in the main article <a href="/wiki/Multiple_sequence_alignment" title="Multiple sequence alignment">multiple sequence alignment</a>.</p>
<h2><span class="mw-headline" id="Structural_alignment">Structural alignment</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=15" title="Edit section: Structural alignment">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Structural_alignment" title="Structural alignment">Structural alignment</a></div>
<p>Structural alignments, which are usually specific to protein and sometimes RNA sequences, use information about the <a href="/wiki/Secondary_structure" class="mw-redirect" title="Secondary structure">secondary</a> and <a href="/wiki/Tertiary_structure" class="mw-redirect" title="Tertiary structure">tertiary structure</a> of the protein or RNA molecule to aid in aligning the sequences. These methods can be used for two or more sequences and typically produce local alignments; however, because they depend on the availability of structural information, they can only be used for sequences whose corresponding structures are known (usually through <a href="/wiki/X-ray_crystallography" title="X-ray crystallography">X-ray crystallography</a> or <a href="/wiki/NMR_spectroscopy" class="mw-redirect" title="NMR spectroscopy">NMR spectroscopy</a>). Because both protein and RNA structure is more evolutionarily conserved than sequence,<sup id="cite_ref-chothia_15-0" class="reference"><a href="#cite_note-chothia-15">[15]</a></sup> structural alignments can be more reliable between sequences that are very distantly related and that have diverged so extensively that sequence comparison cannot reliably detect their similarity.</p>
<p>Structural alignments are used as the "gold standard" in evaluating alignments for homology-based <a href="/wiki/Protein_structure_prediction" title="Protein structure prediction">protein structure prediction</a><sup id="cite_ref-skolnick_16-0" class="reference"><a href="#cite_note-skolnick-16">[16]</a></sup> because they explicitly align regions of the protein sequence that are structurally similar rather than relying exclusively on sequence information. However, clearly structural alignments cannot be used in structure prediction because at least one sequence in the query set is the target to be modeled, for which the structure is not known. It has been shown that, given the structural alignment between a target and a template sequence, highly accurate models of the target protein sequence can be produced; a major stumbling block in homology-based structure prediction is the production of structurally accurate alignments given only sequence information.<sup id="cite_ref-skolnick_16-1" class="reference"><a href="#cite_note-skolnick-16">[16]</a></sup></p>
<h3><span class="mw-headline" id="DALI">DALI</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=16" title="Edit section: DALI">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The DALI method, or <a href="/wiki/Distance_matrix" title="Distance matrix">distance matrix</a> alignment, is a fragment-based method for constructing structural alignments based on contact similarity patterns between successive hexapeptides in the query sequences.<sup id="cite_ref-holm_17-0" class="reference"><a href="#cite_note-holm-17">[17]</a></sup> It can generate pairwise or multiple alignments and identify a query sequence's structural neighbors in the <a href="/wiki/Protein_Data_Bank" title="Protein Data Bank">Protein Data Bank</a> (PDB). It has been used to construct the <a href="/wiki/Families_of_structurally_similar_proteins" class="mw-redirect" title="Families of structurally similar proteins">FSSP</a> structural alignment database (Fold classification based on Structure-Structure alignment of Proteins, or Families of Structurally Similar Proteins). A DALI webserver can be accessed at <a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/dali/">EBI DALI</a> and the FSSP is located at <a rel="nofollow" class="external text" href="http://ekhidna.biocenter.helsinki.fi/dali/start">The Dali Database</a>.</p>
<h3><span class="mw-headline" id="SSAP">SSAP</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=17" title="Edit section: SSAP">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>SSAP (sequential structure alignment program) is a dynamic programming-based method of structural alignment that uses atom-to-atom vectors in structure space as comparison points. It has been extended since its original description to include multiple as well as pairwise alignments,<sup id="cite_ref-taylor_18-0" class="reference"><a href="#cite_note-taylor-18">[18]</a></sup> and has been used in the construction of the <a href="/wiki/CATH" class="mw-redirect" title="CATH">CATH</a> (Class, Architecture, Topology, Homology) hierarchical database classification of protein folds.<sup id="cite_ref-orengo_19-0" class="reference"><a href="#cite_note-orengo-19">[19]</a></sup> The CATH database can be accessed at <a rel="nofollow" class="external text" href="http://www.cathdb.info/">CATH Protein Structure Classification</a>.</p>
<h3><span class="mw-headline" id="Combinatorial_extension">Combinatorial extension</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=18" title="Edit section: Combinatorial extension">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The combinatorial extension method of structural alignment generates a pairwise structural alignment by using local geometry to align short fragments of the two proteins being analyzed and then assembles these fragments into a larger alignment.<sup id="cite_ref-shindyalov_20-0" class="reference"><a href="#cite_note-shindyalov-20">[20]</a></sup> Based on measures such as rigid-body <a href="/wiki/Root_mean_square_deviation_(bioinformatics)" class="mw-redirect" title="Root mean square deviation (bioinformatics)">root mean square distance</a>, residue distances, local secondary structure, and surrounding environmental features such as residue neighbor <a href="/wiki/Hydrophobic" class="mw-redirect" title="Hydrophobic">hydrophobicity</a>, local alignments called "aligned fragment pairs" are generated and used to build a similarity matrix representing all possible structural alignments within predefined cutoff criteria. A path from one protein structure state to the other is then traced through the matrix by extending the growing alignment one fragment at a time. The optimal such path defines the combinatorial-extension alignment. A web-based server implementing the method and providing a database of pairwise alignments of structures in the Protein Data Bank is located at the <a rel="nofollow" class="external text" href="http://web.archive.org/web/cl.sdsc.edu/">Combinatorial Extension</a> website.</p>
<h2><span class="mw-headline" id="Phylogenetic_analysis">Phylogenetic analysis</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=19" title="Edit section: Phylogenetic analysis">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Computational_phylogenetics" title="Computational phylogenetics">Computational phylogenetics</a></div>
<p>Phylogenetics and sequence alignment are closely related fields due to the shared necessity of evaluating sequence relatedness.<sup id="cite_ref-ortet_21-0" class="reference"><a href="#cite_note-ortet-21">[21]</a></sup> The field of <a href="/wiki/Phylogenetics" title="Phylogenetics">phylogenetics</a> makes extensive use of sequence alignments in the construction and interpretation of <a href="/wiki/Phylogenetic_tree" title="Phylogenetic tree">phylogenetic trees</a>, which are used to classify the evolutionary relationships between homologous <a href="/wiki/Gene" title="Gene">genes</a> represented in the <a href="/wiki/Genome" title="Genome">genomes</a> of divergent species. The degree to which sequences in a query set differ is qualitatively related to the sequences' evolutionary distance from one another. Roughly speaking, high sequence identity suggests that the sequences in question have a comparatively young <a href="/wiki/Most_recent_common_ancestor" title="Most recent common ancestor">most recent common ancestor</a>, while low identity suggests that the divergence is more ancient. This approximation, which reflects the "<a href="/wiki/Molecular_clock" title="Molecular clock">molecular clock</a>" hypothesis that a roughly constant rate of evolutionary change can be used to extrapolate the elapsed time since two genes first diverged (that is, the <a href="/wiki/Coalescence_(genetics)" class="mw-redirect" title="Coalescence (genetics)">coalescence</a> time), assumes that the effects of mutation and <a href="/wiki/Natural_selection" title="Natural selection">selection</a> are constant across sequence lineages. Therefore it does not account for possible difference among organisms or species in the rates of <a href="/wiki/DNA_repair" title="DNA repair">DNA repair</a> or the possible functional conservation of specific regions in a sequence. (In the case of nucleotide sequences, the molecular clock hypothesis in its most basic form also discounts the difference in acceptance rates between <a href="/wiki/Silent_mutation" title="Silent mutation">silent mutations</a> that do not alter the meaning of a given <a href="/wiki/Codon" class="mw-redirect" title="Codon">codon</a> and other mutations that result in a different <a href="/wiki/Amino_acid" title="Amino acid">amino acid</a> being incorporated into the protein.) More statistically accurate methods allow the evolutionary rate on each branch of the phylogenetic tree to vary, thus producing better estimates of coalescence times for genes.</p>
<p>Progressive multiple alignment techniques produce a phylogenetic tree by necessity because they incorporate sequences into the growing alignment in order of relatedness. Other techniques that assemble multiple sequence alignments and phylogenetic trees score and sort trees first and calculate a multiple sequence alignment from the highest-scoring tree. Commonly used methods of phylogenetic tree construction are mainly <a href="/wiki/Heuristic" title="Heuristic">heuristic</a> because the problem of selecting the optimal tree, like the problem of selecting the optimal multiple sequence alignment, is <a href="/wiki/NP-hard" class="mw-redirect" title="NP-hard">NP-hard</a>.<sup id="cite_ref-felsenstein_22-0" class="reference"><a href="#cite_note-felsenstein-22">[22]</a></sup></p>
<h3><span class="mw-headline" id="Assessment_of_significance">Assessment of significance</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=20" title="Edit section: Assessment of significance">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Sequence alignments are useful in bioinformatics for identifying sequence similarity, producing phylogenetic trees, and developing homology models of protein structures. However, the biological relevance of sequence alignments is not always clear. Alignments are often assumed to reflect a degree of evolutionary change between sequences descended from a common ancestor; however, it is formally possible that <a href="/wiki/Convergent_evolution" title="Convergent evolution">convergent evolution</a> can occur to produce apparent similarity between proteins that are evolutionarily unrelated but perform similar functions and have similar structures.</p>
<p>In database searches such as BLAST, statistical methods can determine the likelihood of a particular alignment between sequences or sequence regions arising by chance given the size and composition of the database being searched. These values can vary significantly depending on the search space. In particular, the likelihood of finding a given alignment by chance increases if the database consists only of sequences from the same organism as the query sequence. Repetitive sequences in the database or query can also distort both the search results and the assessment of statistical significance; BLAST automatically filters such repetitive sequences in the query to avoid apparent hits that are statistical artifacts.</p>
<p>Methods of statistical significance estimation for gapped sequence alignments are available in the literature.<sup id="cite_ref-ortet_21-1" class="reference"><a href="#cite_note-ortet-21">[21]</a></sup><sup id="cite_ref-altschul_23-0" class="reference"><a href="#cite_note-altschul-23">[23]</a></sup><sup id="cite_ref-hartmann_24-0" class="reference"><a href="#cite_note-hartmann-24">[24]</a></sup><sup id="cite_ref-newberg_25-0" class="reference"><a href="#cite_note-newberg-25">[25]</a></sup><sup id="cite_ref-eddy_26-0" class="reference"><a href="#cite_note-eddy-26">[26]</a></sup><sup id="cite_ref-bastien_27-0" class="reference"><a href="#cite_note-bastien-27">[27]</a></sup></p>
<h3><span class="mw-headline" id="Assessment_of_credibility">Assessment of credibility</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=21" title="Edit section: Assessment of credibility">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Statistical significance indicates the probability that an alignment of a given quality could arise by chance, but does not indicate how much superior a given alignment is to alternative alignments of the same sequences. Measures of alignment credibility indicate the extent to which the best scoring alignments for a given pair of sequences are substantially similar. Methods of alignment credibility estimation for gapped sequence alignments are available in the literature.<sup id="cite_ref-NewbergLawrence2009_28-0" class="reference"><a href="#cite_note-NewbergLawrence2009-28">[28]</a></sup></p>
<h3><span class="mw-headline" id="Scoring_functions">Scoring functions</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=22" title="Edit section: Scoring functions">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The choice of a scoring function that reflects biological or statistical observations about known sequences is important to producing good alignments. Protein sequences are frequently aligned using <a href="/wiki/Substitution_matrix" title="Substitution matrix">substitution matrices</a> that reflect the probabilities of given character-to-character substitutions. A series of matrices called <a href="/wiki/Point_accepted_mutation" title="Point accepted mutation">PAM matrices</a> (Point Accepted Mutation matrices, originally defined by <a href="/wiki/Margaret_Dayhoff" class="mw-redirect" title="Margaret Dayhoff">Margaret Dayhoff</a> and sometimes referred to as "Dayhoff matrices") explicitly encode evolutionary approximations regarding the rates and probabilities of particular amino acid mutations. Another common series of scoring matrices, known as <a href="/wiki/BLOSUM" title="BLOSUM">BLOSUM</a> (Blocks Substitution Matrix), encodes empirically derived substitution probabilities. Variants of both types of matrices are used to detect sequences with differing levels of divergence, thus allowing users of BLAST or FASTA to restrict searches to more closely related matches or expand to detect more divergent sequences. <a href="/wiki/Gap_penalty" title="Gap penalty">Gap penalties</a> account for the introduction of a gap - on the evolutionary model, an insertion or deletion mutation - in both nucleotide and protein sequences, and therefore the penalty values should be proportional to the expected rate of such mutations. The quality of the alignments produced therefore depends on the quality of the scoring function.</p>
<p>It can be very useful and instructive to try the same alignment several times with different choices for scoring matrix and/or gap penalty values and compare the results. Regions where the solution is weak or non-unique can often be identified by observing which regions of the alignment are robust to variations in alignment parameters.</p>
<h2><span class="mw-headline" id="Other_biological_uses">Other biological uses</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=23" title="Edit section: Other biological uses">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Sequenced RNA, such as <a href="/wiki/Expressed_sequence_tags" class="mw-redirect" title="Expressed sequence tags">expressed sequence tags</a> and full-length mRNAs, can be aligned to a sequenced genome to find where there are genes and get information about <a href="/wiki/Alternative_splicing" title="Alternative splicing">alternative splicing</a><sup id="cite_ref-29" class="reference"><a href="#cite_note-29">[29]</a></sup> and <a href="/wiki/RNA_editing" title="RNA editing">RNA editing</a>.<sup id="cite_ref-30" class="reference"><a href="#cite_note-30">[30]</a></sup> Sequence alignment is also a part of <a href="/wiki/Genome_assembly" class="mw-redirect" title="Genome assembly">genome assembly</a>, where sequences are aligned to find overlap so that <i><a href="/wiki/Contig" title="Contig">contigs</a></i> (long stretches of sequence) can be formed.<sup id="cite_ref-31" class="reference"><a href="#cite_note-31">[31]</a></sup> Another use is <a href="/wiki/Single_nucleotide_polymorphism" class="mw-redirect" title="Single nucleotide polymorphism">SNP</a> analysis, where sequences from different individuals are aligned to find single basepairs that are often different in a population.<sup id="cite_ref-32" class="reference"><a href="#cite_note-32">[32]</a></sup></p>
<h2><span class="mw-headline" id="Non-biological_uses">Non-biological uses</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=24" title="Edit section: Non-biological uses">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>The methods used for biological sequence alignment have also found applications in other fields, most notably in <a href="/wiki/Natural_language_processing" title="Natural language processing">natural language processing</a> and in social sciences, where the <a href="/wiki/Needleman-Wunsch_algorithm" class="mw-redirect" title="Needleman-Wunsch algorithm">Needleman-Wunsch algorithm</a> is usually referred to as <a href="/wiki/Optimal_matching" title="Optimal matching">Optimal matching</a>.<sup id="cite_ref-33" class="reference"><a href="#cite_note-33">[33]</a></sup> Techniques that generate the set of elements from which words will be selected in natural-language generation algorithms have borrowed multiple sequence alignment techniques from bioinformatics to produce linguistic versions of computer-generated mathematical proofs.<sup id="cite_ref-Barzilay_34-0" class="reference"><a href="#cite_note-Barzilay-34">[34]</a></sup> In the field of historical and comparative <a href="/wiki/Linguistics" title="Linguistics">linguistics</a>, sequence alignment has been used to partially automate the <a href="/wiki/Comparative_method" class="mw-redirect" title="Comparative method">comparative method</a> by which linguists traditionally reconstruct languages.<sup id="cite_ref-35" class="reference"><a href="#cite_note-35">[35]</a></sup> Business and marketing research has also applied multiple sequence alignment techniques in analyzing series of purchases over time.<sup id="cite_ref-prinzie_36-0" class="reference"><a href="#cite_note-prinzie-36">[36]</a></sup></p>
<h2><span class="mw-headline" id="Software">Software</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=25" title="Edit section: Software">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Sequence_alignment_software" class="mw-redirect" title="Sequence alignment software">Sequence alignment software</a></div>
<p>A more complete list of available software categorized by algorithm and alignment type is available at <a href="/wiki/Sequence_alignment_software" class="mw-redirect" title="Sequence alignment software">sequence alignment software</a>, but common software tools used for general sequence alignment tasks include <a rel="nofollow" class="external text" href="http://www2.ebi.ac.uk/clustalw/">ClustalW</a> and <a rel="nofollow" class="external text" href="http://tcoffee.vital-it.ch/cgi-bin/Tcoffee/tcoffee_cgi/index.cgi">T-coffee</a> for alignment, and <a rel="nofollow" class="external text" href="http://ncbi.nih.gov/BLAST/">BLAST</a> and <a rel="nofollow" class="external text" href="http://fasta.bioch.virginia.edu/fasta_www2/fasta_list2.shtml">FASTA3x</a> for database searching.</p>
<p>Alignment algorithms and software can be directly compared to one another using a standardized set of <a href="/wiki/Benchmark_(computing)" title="Benchmark (computing)">benchmark</a> reference multiple sequence alignments known as BAliBASE.<sup id="cite_ref-thompson2_37-0" class="reference"><a href="#cite_note-thompson2-37">[37]</a></sup> The data set consists of structural alignments, which can be considered a standard against which purely sequence-based methods are compared. The relative performance of many common alignment methods on frequently encountered alignment problems has been tabulated and selected results published online at <a rel="nofollow" class="external text" href="http://bips.u-strasbg.fr/fr/Products/Databases/BAliBASE/prog_scores.html">BAliBASE</a>.<sup id="cite_ref-thompson3_38-0" class="reference"><a href="#cite_note-thompson3-38">[38]</a></sup> A comprehensive list of BAliBASE scores for many (currently 12) different alignment tools can be computed within the protein workbench <a rel="nofollow" class="external text" href="http://3d-alignment.eu/">STRAP</a>.</p>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=26" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li><a href="/wiki/Sequence_mining" class="mw-redirect" title="Sequence mining">Sequence mining</a></li>
<li><a href="/wiki/BLAST" title="BLAST">BLAST</a></li>
</ul>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Sequence_alignment&amp;action=edit&amp;section=27" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="reflist columns references-column-width" style="-moz-column-width: 30em; -webkit-column-width: 30em; column-width: 30em; list-style-type: decimal;">
<ol class="references">
<li id="cite_note-mount-1"><span class="mw-cite-backlink">^ <a href="#cite_ref-mount_1-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-mount_1-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-mount_1-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation book">Mount DM. (2004). <i>Bioinformatics: Sequence and Genome Analysis</i> (2nd ed.). Cold Spring Harbor Laboratory Press: Cold Spring Harbor, NY. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/0-87969-608-7" title="Special:BookSources/0-87969-608-7">0-87969-608-7</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.au=Mount+DM.&amp;rft.btitle=Bioinformatics%3A+Sequence+and+Genome+Analysis&amp;rft.date=2004&amp;rft.edition=2nd&amp;rft.genre=book&amp;rft.isbn=0-87969-608-7&amp;rft.pub=Cold+Spring+Harbor+Laboratory+Press%3A+Cold+Spring+Harbor%2C+NY.&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-2"><span class="mw-cite-backlink"><b><a href="#cite_ref-2">^</a></b></span> <span class="reference-text">ClustalW2 FAQs <a rel="nofollow" class="external free" href="http://www.ebi.ac.uk/Tools/clustalw2/help.html#color">http://www.ebi.ac.uk/Tools/clustalw2/help.html#color</a></span></li>
<li id="cite_note-predict-3"><span class="mw-cite-backlink"><b><a href="#cite_ref-predict_3-0">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://www.ncbi.nlm.nih.gov/pubmed/11337480">Ng PC, Henikoff S. Predicting deleterious amino acid substitutions. Genome Res. 2001 May;11(5):863-74.</a></span></li>
<li id="cite_note-Schneider-4"><span class="mw-cite-backlink"><b><a href="#cite_ref-Schneider_4-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Schneider TD, Stephens RM (1990). <a rel="nofollow" class="external text" href="http://nar.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=2172928">"Sequence logos: a new way to display consensus sequences"</a>. <i>Nucleic Acids Res</i>. <b>18</b> (20): 6097–6100. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC332411">332411</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/2172928">2172928</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fnar%2F18.20.6097">10.1093/nar/18.20.6097</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Sequence+logos%3A+a+new+way+to+display+consensus+sequences&amp;rft.au=Schneider+TD%2C+Stephens+RM&amp;rft.date=1990&amp;rft.genre=article&amp;rft.issue=20&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=6097-6100&amp;rft.volume=18&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC332411&amp;rft_id=http%3A%2F%2Fnar.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D2172928&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F18.20.6097&amp;rft_id=info%3Apmid%2F2172928&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-brudno-5"><span class="mw-cite-backlink"><b><a href="#cite_ref-brudno_5-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Brudno M, Malde S, Poliakov A, Do CB, Couronne O, Dubchak I, Batzoglou S (2003). <a rel="nofollow" class="external text" href="http://bioinformatics.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=12855437">"Glocal alignment: finding rearrangements during alignment"</a>. <i>Bioinformatics</i>. 19. Suppl 1 (90001): i54–62. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12855437">12855437</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fbioinformatics%2Fbtg1005">10.1093/bioinformatics/btg1005</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Glocal+alignment%3A+finding+rearrangements+during+alignment&amp;rft.au=Brudno+M%2C+Malde+S%2C+Poliakov+A%2C+Do+CB%2C+Couronne+O%2C+Dubchak+I%2C+Batzoglou+S&amp;rft.date=2003&amp;rft.genre=article&amp;rft.issue=90001&amp;rft.jtitle=Bioinformatics&amp;rft.pages=i54-62&amp;rft.volume=Suppl+1&amp;rft_id=http%3A%2F%2Fbioinformatics.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D12855437&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtg1005&amp;rft_id=info%3Apmid%2F12855437&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-wang-6"><span class="mw-cite-backlink"><b><a href="#cite_ref-wang_6-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Wang L, Jiang T. (1994). <a rel="nofollow" class="external text" href="http://www.liebertonline.com/doi/abs/10.1089/cmb.1994.1.337">"On the complexity of multiple sequence alignment"</a>. <i>J Comput Biol</i>. <b>1</b> (4): 337–48. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/8790475">8790475</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1089%2Fcmb.1994.1.337">10.1089/cmb.1994.1.337</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=On+the+complexity+of+multiple+sequence+alignment&amp;rft.au=Wang+L%2C+Jiang+T.&amp;rft.date=1994&amp;rft.genre=article&amp;rft.issue=4&amp;rft.jtitle=J+Comput+Biol&amp;rft.pages=337-48&amp;rft.volume=1&amp;rft_id=http%3A%2F%2Fwww.liebertonline.com%2Fdoi%2Fabs%2F10.1089%2Fcmb.1994.1.337&amp;rft_id=info%3Adoi%2F10.1089%2Fcmb.1994.1.337&amp;rft_id=info%3Apmid%2F8790475&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-elias-7"><span class="mw-cite-backlink"><b><a href="#cite_ref-elias_7-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Elias, Isaac (2006). <a rel="nofollow" class="external text" href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2006.13.1323">"Settling the intractability of multiple alignment"</a>. <i>J Comput Biol</i>. <b>13</b> (7): 1323–1339. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/17037961">17037961</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1089%2Fcmb.2006.13.1323">10.1089/cmb.2006.13.1323</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Settling+the+intractability+of+multiple+alignment&amp;rft.au=Elias%2C+Isaac&amp;rft.date=2006&amp;rft.genre=article&amp;rft.issue=7&amp;rft.jtitle=J+Comput+Biol&amp;rft.pages=1323-1339&amp;rft.volume=13&amp;rft_id=http%3A%2F%2Fwww.liebertonline.com%2Fdoi%2Fabs%2F10.1089%2Fcmb.2006.13.1323&amp;rft_id=info%3Adoi%2F10.1089%2Fcmb.2006.13.1323&amp;rft_id=info%3Apmid%2F17037961&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-lipman-8"><span class="mw-cite-backlink"><b><a href="#cite_ref-lipman_8-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Lipman DJ, Altschul SF, Kececioglu JD (1989). <a rel="nofollow" class="external text" href="http://www.pnas.org/cgi/pmidlookup?view=long&amp;pmid=2734293">"A tool for multiple sequence alignment"</a>. <i>Proc Natl Acad Sci USA</i>. <b>86</b> (12): 4412–5. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC287279">287279</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/2734293">2734293</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1073%2Fpnas.86.12.4412">10.1073/pnas.86.12.4412</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=A+tool+for+multiple+sequence+alignment&amp;rft.au=Lipman+DJ%2C+Altschul+SF%2C+Kececioglu+JD&amp;rft.date=1989&amp;rft.genre=article&amp;rft.issue=12&amp;rft.jtitle=Proc+Natl+Acad+Sci+USA&amp;rft.pages=4412-5&amp;rft.volume=86&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC287279&amp;rft_id=http%3A%2F%2Fwww.pnas.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D2734293&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.86.12.4412&amp;rft_id=info%3Apmid%2F2734293&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-higgins-9"><span class="mw-cite-backlink"><b><a href="#cite_ref-higgins_9-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Higgins DG, Sharp PM (1988). <a rel="nofollow" class="external text" href="http://linkinghub.elsevier.com/retrieve/pii/0378-1119(88)90330-7">"CLUSTAL: a package for performing multiple sequence alignment on a microcomputer"</a>. <i>Gene</i>. <b>73</b> (1): 237–44. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/3243435">3243435</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2F0378-1119%2888%2990330-7">10.1016/0378-1119(88)90330-7</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=CLUSTAL%3A+a+package+for+performing+multiple+sequence+alignment+on+a+microcomputer&amp;rft.au=Higgins+DG%2C+Sharp+PM&amp;rft.date=1988&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Gene&amp;rft.pages=237-44&amp;rft.volume=73&amp;rft_id=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2F0378-1119%2888%2990330-7&amp;rft_id=info%3Adoi%2F10.1016%2F0378-1119%2888%2990330-7&amp;rft_id=info%3Apmid%2F3243435&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-thompson-10"><span class="mw-cite-backlink"><b><a href="#cite_ref-thompson_10-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Thompson JD, Higgins DG, Gibson TJ. (1994). <a rel="nofollow" class="external text" href="http://nar.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=7984417">"CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice"</a>. <i>Nucleic Acids Res</i>. <b>22</b> (22): 4673–80. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC308517">308517</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/7984417">7984417</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fnar%2F22.22.4673">10.1093/nar/22.22.4673</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice&amp;rft.au=Thompson+JD%2C+Higgins+DG%2C+Gibson+TJ.&amp;rft.date=1994&amp;rft.genre=article&amp;rft.issue=22&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=4673-80&amp;rft.volume=22&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC308517&amp;rft_id=http%3A%2F%2Fnar.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D7984417&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F22.22.4673&amp;rft_id=info%3Apmid%2F7984417&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span><sup class="noprint Inline-Template"><span style="white-space: nowrap;">[<i><a href="/wiki/Wikipedia:Link_rot" title="Wikipedia:Link rot"><span title="&#160;Dead link since May 2010">dead link</span></a></i>]</span></sup></span></li>
<li id="cite_note-chenna-11"><span class="mw-cite-backlink"><b><a href="#cite_ref-chenna_11-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Chenna R, Sugawara H, Koike T, Lopez R, Gibson TJ, Higgins DG, Thompson JD. (2003). <a rel="nofollow" class="external text" href="http://nar.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=12824352">"Multiple sequence alignment with the Clustal series of programs"</a>. <i>Nucleic Acids Res</i>. <b>31</b> (13): 3497–500. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC168907">168907</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12824352">12824352</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fnar%2Fgkg500">10.1093/nar/gkg500</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Multiple+sequence+alignment+with+the+Clustal+series+of+programs&amp;rft.au=Chenna+R%2C+Sugawara+H%2C+Koike+T%2C+Lopez+R%2C+Gibson+TJ%2C+Higgins+DG%2C+Thompson+JD.&amp;rft.date=2003&amp;rft.genre=article&amp;rft.issue=13&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=3497-500&amp;rft.volume=31&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC168907&amp;rft_id=http%3A%2F%2Fnar.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D12824352&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkg500&amp;rft_id=info%3Apmid%2F12824352&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-notredame-12"><span class="mw-cite-backlink"><b><a href="#cite_ref-notredame_12-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Notredame C, Higgins DG, Heringa J. (2000). <a rel="nofollow" class="external text" href="http://linkinghub.elsevier.com/retrieve/pii/S0022-2836(00)94042-7">"T-Coffee: A novel method for fast and accurate multiple sequence alignment"</a>. <i>J Mol Biol</i>. <b>302</b> (1): 205–17. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/10964570">10964570</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1006%2Fjmbi.2000.4042">10.1006/jmbi.2000.4042</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=T-Coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment&amp;rft.au=Notredame+C%2C+Higgins+DG%2C+Heringa+J.&amp;rft.date=2000&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=J+Mol+Biol&amp;rft.pages=205-17&amp;rft.volume=302&amp;rft_id=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0022-2836%2800%2994042-7&amp;rft_id=info%3Adoi%2F10.1006%2Fjmbi.2000.4042&amp;rft_id=info%3Apmid%2F10964570&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-hirosawa-13"><span class="mw-cite-backlink"><b><a href="#cite_ref-hirosawa_13-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hirosawa M, Totoki Y, Hoshida M, Ishikawa M. (1995). <a rel="nofollow" class="external text" href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/11/1/13">"Comprehensive study on iterative algorithms of multiple sequence alignment"</a>. <i>Comput Appl Biosci</i>. <b>11</b> (1): 13–8. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/7796270">7796270</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fbioinformatics%2F11.1.13">10.1093/bioinformatics/11.1.13</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Comprehensive+study+on+iterative+algorithms+of+multiple+sequence+alignment&amp;rft.au=Hirosawa+M%2C+Totoki+Y%2C+Hoshida+M%2C+Ishikawa+M.&amp;rft.date=1995&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Comput+Appl+Biosci&amp;rft.pages=13-8&amp;rft.volume=11&amp;rft_id=http%3A%2F%2Fbioinformatics.oxfordjournals.org%2Fcgi%2Fcontent%2Fabstract%2F11%2F1%2F13&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F11.1.13&amp;rft_id=info%3Apmid%2F7796270&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-karplus-14"><span class="mw-cite-backlink"><b><a href="#cite_ref-karplus_14-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Karplus K, Barrett C, Hughey R. (1998). <a rel="nofollow" class="external text" href="http://bioinformatics.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=9927713">"Hidden Markov models for detecting remote protein homologies"</a>. <i>Bioinformatics</i>. <b>14</b> (10): 846–856. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/9927713">9927713</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fbioinformatics%2F14.10.846">10.1093/bioinformatics/14.10.846</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Hidden+Markov+models+for+detecting+remote+protein+homologies&amp;rft.au=Karplus+K%2C+Barrett+C%2C+Hughey+R.&amp;rft.date=1998&amp;rft.genre=article&amp;rft.issue=10&amp;rft.jtitle=Bioinformatics&amp;rft.pages=846-856&amp;rft.volume=14&amp;rft_id=http%3A%2F%2Fbioinformatics.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D9927713&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F14.10.846&amp;rft_id=info%3Apmid%2F9927713&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-chothia-15"><span class="mw-cite-backlink"><b><a href="#cite_ref-chothia_15-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Chothia C, Lesk AM. (1986). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1166865">"The relation between the divergence of sequence and structure in proteins"</a>. <i>EMBO J</i>. <b>5</b> (4): 823–6. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1166865">1166865</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/3709526">3709526</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=The+relation+between+the+divergence+of+sequence+and+structure+in+proteins&amp;rft.au=Chothia+C%2C+Lesk+AM.&amp;rft.date=1986&amp;rft.genre=article&amp;rft.issue=4&amp;rft.jtitle=EMBO+J&amp;rft.pages=823-6&amp;rft.volume=5&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1166865&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1166865&amp;rft_id=info%3Apmid%2F3709526&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span> <span style="font-size:100%" class="error citation-comment">Unknown parameter <code style="color:inherit; border:inherit; padding:inherit;">|month=</code> ignored (<a href="/wiki/Help:CS1_errors#parameter_ignored" title="Help:CS1 errors">help</a>)</span></span></li>
<li id="cite_note-skolnick-16"><span class="mw-cite-backlink">^ <a href="#cite_ref-skolnick_16-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-skolnick_16-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Zhang Y, Skolnick J. (2005). <a rel="nofollow" class="external text" href="http://www.pnas.org/cgi/pmidlookup?view=long&amp;pmid=15653774">"The protein structure prediction problem could be solved using the current PDB library"</a>. <i>Proc Natl Acad Sci USA</i>. <b>102</b> (4): 1029–34. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC545829">545829</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15653774">15653774</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1073%2Fpnas.0407152101">10.1073/pnas.0407152101</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=The+protein+structure+prediction+problem+could+be+solved+using+the+current+PDB+library&amp;rft.au=Zhang+Y%2C+Skolnick+J.&amp;rft.date=2005&amp;rft.genre=article&amp;rft.issue=4&amp;rft.jtitle=Proc+Natl+Acad+Sci+USA&amp;rft.pages=1029-34&amp;rft.volume=102&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC545829&amp;rft_id=http%3A%2F%2Fwww.pnas.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D15653774&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0407152101&amp;rft_id=info%3Apmid%2F15653774&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-holm-17"><span class="mw-cite-backlink"><b><a href="#cite_ref-holm_17-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Holm L, Sander C (1996). <a rel="nofollow" class="external text" href="http://www.sciencemag.org/cgi/pmidlookup?view=long&amp;pmid=8662544">"Mapping the protein universe"</a>. <i>Science</i>. <b>273</b> (5275): 595–603. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/8662544">8662544</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1126%2Fscience.273.5275.595">10.1126/science.273.5275.595</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Mapping+the+protein+universe&amp;rft.au=Holm+L%2C+Sander+C&amp;rft.date=1996&amp;rft.genre=article&amp;rft.issue=5275&amp;rft.jtitle=Science&amp;rft.pages=595-603&amp;rft.volume=273&amp;rft_id=http%3A%2F%2Fwww.sciencemag.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D8662544&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.273.5275.595&amp;rft_id=info%3Apmid%2F8662544&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-taylor-18"><span class="mw-cite-backlink"><b><a href="#cite_ref-taylor_18-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Taylor WR, Flores TP, Orengo CA. (1994). <a rel="nofollow" class="external text" href="http://www.proteinscience.org/cgi/pmidlookup?view=long&amp;pmid=7849601">"Multiple protein structure alignment"</a>. <i>Protein Sci</i>. <b>3</b> (10): 1858–70. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2142613">2142613</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/7849601">7849601</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1002%2Fpro.5560031025">10.1002/pro.5560031025</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Multiple+protein+structure+alignment&amp;rft.au=Taylor+WR%2C+Flores+TP%2C+Orengo+CA.&amp;rft.date=1994&amp;rft.genre=article&amp;rft.issue=10&amp;rft.jtitle=Protein+Sci&amp;rft.pages=1858-70&amp;rft.volume=3&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2142613&amp;rft_id=http%3A%2F%2Fwww.proteinscience.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D7849601&amp;rft_id=info%3Adoi%2F10.1002%2Fpro.5560031025&amp;rft_id=info%3Apmid%2F7849601&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-orengo-19"><span class="mw-cite-backlink"><b><a href="#cite_ref-orengo_19-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Orengo CA, Michie AD, Jones S, Jones DT, Swindells MB, Thornton JM (1997). "CATH--a hierarchic classification of protein domain structures". <i>Structure</i>. <b>5</b> (8): 1093–108. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/9309224">9309224</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2FS0969-2126%2897%2900260-8">10.1016/S0969-2126(97)00260-8</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=CATH--a+hierarchic+classification+of+protein+domain+structures&amp;rft.au=Orengo+CA%2C+Michie+AD%2C+Jones+S%2C+Jones+DT%2C+Swindells+MB%2C+Thornton+JM&amp;rft.date=1997&amp;rft.genre=article&amp;rft.issue=8&amp;rft.jtitle=Structure&amp;rft.pages=1093-108&amp;rft.volume=5&amp;rft_id=info%3Adoi%2F10.1016%2FS0969-2126%2897%2900260-8&amp;rft_id=info%3Apmid%2F9309224&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-shindyalov-20"><span class="mw-cite-backlink"><b><a href="#cite_ref-shindyalov_20-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Shindyalov IN, Bourne PE. (1998). <a rel="nofollow" class="external text" href="http://peds.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=9796821">"Protein structure alignment by incremental combinatorial extension (CE) of the optimal path"</a>. <i>Protein Eng</i>. <b>11</b> (9): 739–47. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/9796821">9796821</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fprotein%2F11.9.739">10.1093/protein/11.9.739</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Protein+structure+alignment+by+incremental+combinatorial+extension+%28CE%29+of+the+optimal+path&amp;rft.au=Shindyalov+IN%2C+Bourne+PE.&amp;rft.date=1998&amp;rft.genre=article&amp;rft.issue=9&amp;rft.jtitle=Protein+Eng&amp;rft.pages=739-47&amp;rft.volume=11&amp;rft_id=http%3A%2F%2Fpeds.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D9796821&amp;rft_id=info%3Adoi%2F10.1093%2Fprotein%2F11.9.739&amp;rft_id=info%3Apmid%2F9796821&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-ortet-21"><span class="mw-cite-backlink">^ <a href="#cite_ref-ortet_21-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-ortet_21-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Ortet P, Bastien O (2010). <a rel="nofollow" class="external text" href="http://www.la-press.com/where-does-the-alignment-score-distribution-shape-come-from-article-a2393">"Where Does the Alignment Score Distribution Shape Come from?"</a>. <i>Evolutionary Bioinformatics</i>. <b>6</b>: 159–187. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21258650">21258650</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.4137%2FEBO.S5875">10.4137/EBO.S5875</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Where+Does+the+Alignment+Score+Distribution+Shape+Come+from%3F&amp;rft.au=Ortet+P%2C+Bastien+O&amp;rft.date=2010&amp;rft.genre=article&amp;rft.jtitle=Evolutionary+Bioinformatics&amp;rft.pages=159-187&amp;rft.volume=6&amp;rft_id=http%3A%2F%2Fwww.la-press.com%2Fwhere-does-the-alignment-score-distribution-shape-come-from-article-a2393&amp;rft_id=info%3Adoi%2F10.4137%2FEBO.S5875&amp;rft_id=info%3Apmid%2F21258650&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-felsenstein-22"><span class="mw-cite-backlink"><b><a href="#cite_ref-felsenstein_22-0">^</a></b></span> <span class="reference-text"><cite class="citation book">Felsenstein J. (2004). <i>Inferring Phylogenies</i>. Sinauer Associates: Sunderland, MA. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/0-87893-177-5" title="Special:BookSources/0-87893-177-5">0-87893-177-5</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.au=Felsenstein+J.&amp;rft.btitle=Inferring+Phylogenies&amp;rft.date=2004&amp;rft.genre=book&amp;rft.isbn=0-87893-177-5&amp;rft.pub=Sinauer+Associates%3A+Sunderland%2C+MA&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-altschul-23"><span class="mw-cite-backlink"><b><a href="#cite_ref-altschul_23-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Altschul SF, Gish W (1996). "Local Alignment Statistics". <i>Meth.Enz</i>. <b>266</b>: 460–480. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2FS0076-6879%2896%2966029-7">10.1016/S0076-6879(96)66029-7</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Local+Alignment+Statistics&amp;rft.au=Altschul+SF%2C+Gish+W&amp;rft.date=1996&amp;rft.genre=article&amp;rft.jtitle=Meth.Enz.&amp;rft.pages=460-480&amp;rft.volume=266&amp;rft_id=info%3Adoi%2F10.1016%2FS0076-6879%2896%2966029-7&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><a rel="nofollow" class="external text" href="http://papercore.org/Altschul1996">Papercore summary</a></span></li>
<li id="cite_note-hartmann-24"><span class="mw-cite-backlink"><b><a href="#cite_ref-hartmann_24-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hartmann AK (2002). <a rel="nofollow" class="external text" href="http://link.aps.org/doi/10.1103/PhysRevE.65.056102">"Sampling rare events: statistics of local sequence alignments"</a>. <i>Phys. Rev. E</i>. <b>65</b>: 056102. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1103%2FPhysRevE.65.056102">10.1103/PhysRevE.65.056102</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Sampling+rare+events%3A+statistics+of+local+sequence+alignments&amp;rft.au=Hartmann+AK&amp;rft.date=2002&amp;rft.genre=article&amp;rft.jtitle=Phys.+Rev.+E&amp;rft.pages=056102&amp;rft.volume=65&amp;rft_id=http%3A%2F%2Flink.aps.org%2Fdoi%2F10.1103%2FPhysRevE.65.056102&amp;rft_id=info%3Adoi%2F10.1103%2FPhysRevE.65.056102&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-newberg-25"><span class="mw-cite-backlink"><b><a href="#cite_ref-newberg_25-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Newberg LA (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2737730">"Significance of gapped sequence alignments"</a>. <i>J Comput Biolo</i>. <b>15</b> (9): 1187–1194. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2737730">2737730</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18973434">18973434</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1089%2Fcmb.2008.0125">10.1089/cmb.2008.0125</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Significance+of+gapped+sequence+alignments&amp;rft.au=Newberg+LA&amp;rft.date=2008&amp;rft.genre=article&amp;rft.issue=9&amp;rft.jtitle=J+Comput+Biolo&amp;rft.pages=1187-1194&amp;rft.volume=15&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2737730&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2737730&amp;rft_id=info%3Adoi%2F10.1089%2Fcmb.2008.0125&amp;rft_id=info%3Apmid%2F18973434&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-eddy-26"><span class="mw-cite-backlink"><b><a href="#cite_ref-eddy_26-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Eddy SR; Rost, Burkhard (2008). Rost, Burkhard, ed. <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2396288">"A probabilistic model of local sequence alignment that simplifies statistical significance estimation"</a>. <i>PLoS Comput Biol</i>. <b>4</b> (5): e1000069. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2396288">2396288</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18516236">18516236</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1371%2Fjournal.pcbi.1000069">10.1371/journal.pcbi.1000069</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=A+probabilistic+model+of+local+sequence+alignment+that+simplifies+statistical+significance+estimation&amp;rft.au=Eddy+SR&amp;rft.au=Rost%2C+Burkhard&amp;rft.date=2008&amp;rft.genre=article&amp;rft.issue=5&amp;rft.jtitle=PLoS+Comput+Biol&amp;rft.pages=e1000069&amp;rft.volume=4&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2396288&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2396288&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.1000069&amp;rft_id=info%3Apmid%2F18516236&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-bastien-27"><span class="mw-cite-backlink"><b><a href="#cite_ref-bastien_27-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bastien O, Aude JC, Roy S, Marechal E (2004). <a rel="nofollow" class="external text" href="http://bioinformatics.oxfordjournals.org/content/20/4/534.long">"Fundamentals of massive automatic pairwise alignments of protein sequences: theoretical significance of Z-value statistics"</a>. <i>Bioinformatics</i>. <b>20</b> (4): 534–537. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/14990449">14990449</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fbioinformatics%2Fbtg440">10.1093/bioinformatics/btg440</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Fundamentals+of+massive+automatic+pairwise+alignments+of+protein+sequences%3A+theoretical+significance+of+Z-value+statistics&amp;rft.au=Bastien+O%2C+Aude+JC%2C+Roy+S%2C+Marechal+E&amp;rft.date=2004&amp;rft.genre=article&amp;rft.issue=4&amp;rft.jtitle=Bioinformatics&amp;rft.pages=534-537&amp;rft.volume=20&amp;rft_id=http%3A%2F%2Fbioinformatics.oxfordjournals.org%2Fcontent%2F20%2F4%2F534.long&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtg440&amp;rft_id=info%3Apmid%2F14990449&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-NewbergLawrence2009-28"><span class="mw-cite-backlink"><b><a href="#cite_ref-NewbergLawrence2009_28-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Newberg LA, Lawrence CE (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2858568">"Exact Calculation of Distributions on Integers, with Application to Sequence Alignment"</a>. <i>J Comput Biolo</i>. <b>16</b> (1): 1–18. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2858568">2858568</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/19119992">19119992</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1089%2Fcmb.2008.0137">10.1089/cmb.2008.0137</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Exact+Calculation+of+Distributions+on+Integers%2C+with+Application+to+Sequence+Alignment&amp;rft.au=Newberg+LA%2C+Lawrence+CE&amp;rft.date=2009&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=J+Comput+Biolo&amp;rft.pages=1-18&amp;rft.volume=16&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2858568&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2858568&amp;rft_id=info%3Adoi%2F10.1089%2Fcmb.2008.0137&amp;rft_id=info%3Apmid%2F19119992&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-29"><span class="mw-cite-backlink"><b><a href="#cite_ref-29">^</a></b></span> <span class="reference-text"><cite class="citation journal">Kim N, Lee C (2008). "Bioinformatics detection of alternative splicing". <i>Methods Mol. Biol</i>. <b>452</b>: 179–97. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18566765">18566765</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1007%2F978-1-60327-159-2_9">10.1007/978-1-60327-159-2_9</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Bioinformatics+detection+of+alternative+splicing&amp;rft.au=Kim+N%2C+Lee+C&amp;rft.date=2008&amp;rft.genre=article&amp;rft.jtitle=Methods+Mol.+Biol.&amp;rft.pages=179-97&amp;rft.volume=452&amp;rft_id=info%3Adoi%2F10.1007%2F978-1-60327-159-2_9&amp;rft_id=info%3Apmid%2F18566765&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-30"><span class="mw-cite-backlink"><b><a href="#cite_ref-30">^</a></b></span> <span class="reference-text"><cite class="citation journal">Li JB, Levanon EY, Yoon JK; et al. (2009). "Genome-wide identification of human RNA editing sites by parallel DNA capturing and sequencing". <i>Science</i>. <b>324</b> (5931): 1210–3. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/19478186">19478186</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1126%2Fscience.1170995">10.1126/science.1170995</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Genome-wide+identification+of+human+RNA+editing+sites+by+parallel+DNA+capturing+and+sequencing&amp;rft.au=Li+JB%2C+Levanon+EY%2C+Yoon+JK&amp;rft.date=2009&amp;rft.genre=article&amp;rft.issue=5931&amp;rft.jtitle=Science&amp;rft.pages=1210-3&amp;rft.volume=324&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1170995&amp;rft_id=info%3Apmid%2F19478186&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span> <span style="font-size:100%" class="error citation-comment">Unknown parameter <code style="color:inherit; border:inherit; padding:inherit;">|month=</code> ignored (<a href="/wiki/Help:CS1_errors#parameter_ignored" title="Help:CS1 errors">help</a>)</span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Explicit use of et al. (<a href="/wiki/Category:CS1_maint:_Explicit_use_of_et_al." title="Category:CS1 maint: Explicit use of et al.">link</a>)CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-31"><span class="mw-cite-backlink"><b><a href="#cite_ref-31">^</a></b></span> <span class="reference-text"><cite class="citation journal">Blazewicz J, Bryja M, Figlerowicz M; et al. (2009). "Whole genome assembly from 454 sequencing output via modified DNA graph concept". <i>Comput Biol Chem</i>. <b>33</b> (3): 224–30. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/19477687">19477687</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fj.compbiolchem.2009.04.005">10.1016/j.compbiolchem.2009.04.005</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Whole+genome+assembly+from+454+sequencing+output+via+modified+DNA+graph+concept&amp;rft.au=Blazewicz+J%2C+Bryja+M%2C+Figlerowicz+M&amp;rft.date=2009&amp;rft.genre=article&amp;rft.issue=3&amp;rft.jtitle=Comput+Biol+Chem&amp;rft.pages=224-30&amp;rft.volume=33&amp;rft_id=info%3Adoi%2F10.1016%2Fj.compbiolchem.2009.04.005&amp;rft_id=info%3Apmid%2F19477687&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span> <span style="font-size:100%" class="error citation-comment">Unknown parameter <code style="color:inherit; border:inherit; padding:inherit;">|month=</code> ignored (<a href="/wiki/Help:CS1_errors#parameter_ignored" title="Help:CS1 errors">help</a>)</span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Explicit use of et al. (<a href="/wiki/Category:CS1_maint:_Explicit_use_of_et_al." title="Category:CS1 maint: Explicit use of et al.">link</a>)CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-32"><span class="mw-cite-backlink"><b><a href="#cite_ref-32">^</a></b></span> <span class="reference-text"><cite class="citation journal">Duran C, Appleby N, Vardy M, Imelfort M, Edwards D, Batley J (2009). "Single nucleotide polymorphism discovery in barley using autoSNPdb". <i>Plant Biotechnol. J</i>. <b>7</b> (4): 326–33. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/19386041">19386041</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1111%2Fj.1467-7652.2009.00407.x">10.1111/j.1467-7652.2009.00407.x</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Single+nucleotide+polymorphism+discovery+in+barley+using+autoSNPdb&amp;rft.au=Duran+C%2C+Appleby+N%2C+Vardy+M%2C+Imelfort+M%2C+Edwards+D%2C+Batley+J&amp;rft.date=2009&amp;rft.genre=article&amp;rft.issue=4&amp;rft.jtitle=Plant+Biotechnol.+J.&amp;rft.pages=326-33&amp;rft.volume=7&amp;rft_id=info%3Adoi%2F10.1111%2Fj.1467-7652.2009.00407.x&amp;rft_id=info%3Apmid%2F19386041&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span> <span style="font-size:100%" class="error citation-comment">Unknown parameter <code style="color:inherit; border:inherit; padding:inherit;">|month=</code> ignored (<a href="/wiki/Help:CS1_errors#parameter_ignored" title="Help:CS1 errors">help</a>)</span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-33"><span class="mw-cite-backlink"><b><a href="#cite_ref-33">^</a></b></span> <span class="reference-text"><cite class="citation journal">Abbott A., Tsay A. (2000). "Sequence Analysis and Optimal Matching Methods in Sociology, Review and Prospect". <i>Sociological Methods and Research</i>. <b>29</b> (1): 3–33. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1177%2F0049124100029001001">10.1177/0049124100029001001</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Sequence+Analysis+and+Optimal+Matching+Methods+in+Sociology%2C+Review+and+Prospect&amp;rft.au=Abbott+A.%2C+Tsay+A.&amp;rft.date=2000&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Sociological+Methods+and+Research&amp;rft.pages=3-33&amp;rft.volume=29&amp;rft_id=info%3Adoi%2F10.1177%2F0049124100029001001&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Barzilay-34"><span class="mw-cite-backlink"><b><a href="#cite_ref-Barzilay_34-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Barzilay R, Lee L. (2002). <a rel="nofollow" class="external text" href="http://www.cs.cornell.edu/home/llee/papers/gen-msa.pdf">"Bootstrapping Lexical Choice via Multiple-Sequence Alignment"</a> <span style="font-size:85%;">(PDF)</span>. <i>Proceedings of the Conference on Empirical Methods in Natural Language Processing (EMNLP)</i>. <b>10</b>: 164–171. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.3115%2F1118693.1118715">10.3115/1118693.1118715</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Bootstrapping+Lexical+Choice+via+Multiple-Sequence+Alignment&amp;rft.au=Barzilay+R%2C+Lee+L.&amp;rft.date=2002&amp;rft.genre=article&amp;rft.jtitle=Proceedings+of+the+Conference+on+Empirical+Methods+in+Natural+Language+Processing+%28EMNLP%29&amp;rft.pages=164-171&amp;rft.volume=10&amp;rft_id=http%3A%2F%2Fwww.cs.cornell.edu%2Fhome%2Fllee%2Fpapers%2Fgen-msa.pdf&amp;rft_id=info%3Adoi%2F10.3115%2F1118693.1118715&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-35"><span class="mw-cite-backlink"><b><a href="#cite_ref-35">^</a></b></span> <span class="reference-text"><cite class="citation journal">Kondrak, Grzegorz (2002). <a rel="nofollow" class="external text" href="http://www.cs.ualberta.ca/~kondrak/papers/thesis.pdf">"Algorithms for Language Reconstruction"</a> <span style="font-size:85%;">(PDF)</span>. University of Toronto, Ontario<span class="reference-accessdate">. Retrieved <span class="nowrap">2007-01-21</span></span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Algorithms+for+Language+Reconstruction&amp;rft.au=Kondrak%2C+Grzegorz&amp;rft.date=2002&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Fwww.cs.ualberta.ca%2F~kondrak%2Fpapers%2Fthesis.pdf&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-prinzie-36"><span class="mw-cite-backlink"><b><a href="#cite_ref-prinzie_36-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Prinzie A., D. Van den Poel (2006). <a rel="nofollow" class="external text" href="http://econpapers.repec.org/paper/rugrugwps/05_2F292.htm">"Incorporating sequential information into traditional classification models by using an element/position-sensitive SAM"</a>. <i>Decision Support Systems</i>. <b>42</b> (2): 508–526. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fj.dss.2005.02.004">10.1016/j.dss.2005.02.004</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Incorporating+sequential+information+into+traditional+classification+models+by+using+an+element%2Fposition-sensitive+SAM&amp;rft.au=Prinzie+A.%2C+D.+Van+den+Poel&amp;rft.date=2006&amp;rft.genre=article&amp;rft.issue=2&amp;rft.jtitle=Decision+Support+Systems&amp;rft.pages=508-526&amp;rft.volume=42&amp;rft_id=http%3A%2F%2Feconpapers.repec.org%2Fpaper%2Frugrugwps%2F05_2F292.htm&amp;rft_id=info%3Adoi%2F10.1016%2Fj.dss.2005.02.004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span> See also Prinzie and Van den Poel's paper <cite class="citation journal">Prinzie, A; Vandenpoel, D (2007). <a rel="nofollow" class="external text" href="http://econpapers.repec.org/paper/rugrugwps/07_2F442.htm">"Predicting home-appliance acquisition sequences: Markov/Markov for Discrimination and survival analysis for modeling sequential information in NPTB models"</a>. <i>Decision Support Systems</i>. <b>44</b> (1): 28–45. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fj.dss.2007.02.008">10.1016/j.dss.2007.02.008</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=Predicting+home-appliance+acquisition+sequences%3A+Markov%2FMarkov+for+Discrimination+and+survival+analysis+for+modeling+sequential+information+in+NPTB+models&amp;rft.au=Prinzie%2C+A&amp;rft.au=Vandenpoel%2C+D&amp;rft.date=2007&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Decision+Support+Systems&amp;rft.pages=28-45&amp;rft.volume=44&amp;rft_id=http%3A%2F%2Feconpapers.repec.org%2Fpaper%2Frugrugwps%2F07_2F442.htm&amp;rft_id=info%3Adoi%2F10.1016%2Fj.dss.2007.02.008&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-thompson2-37"><span class="mw-cite-backlink"><b><a href="#cite_ref-thompson2_37-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Thompson JD, Plewniak F, Poch O (1999). <a rel="nofollow" class="external text" href="http://bioinformatics.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=10068696">"BAliBASE: a benchmark alignment database for the evaluation of multiple alignment programs"</a>. <i>Bioinformatics</i>. <b>15</b> (1): 87–8. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/10068696">10068696</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fbioinformatics%2F15.1.87">10.1093/bioinformatics/15.1.87</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=BAliBASE%3A+a+benchmark+alignment+database+for+the+evaluation+of+multiple+alignment+programs&amp;rft.au=Thompson+JD%2C+Plewniak+F%2C+Poch+O&amp;rft.date=1999&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Bioinformatics&amp;rft.pages=87-8&amp;rft.volume=15&amp;rft_id=http%3A%2F%2Fbioinformatics.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D10068696&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F15.1.87&amp;rft_id=info%3Apmid%2F10068696&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-thompson3-38"><span class="mw-cite-backlink"><b><a href="#cite_ref-thompson3_38-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Thompson JD, Plewniak F, Poch O. (1999). <a rel="nofollow" class="external text" href="http://nar.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=10373585">"A comprehensive comparison of multiple sequence alignment programs"</a>. <i>Nucleic Acids Res</i>. <b>27</b> (13): 2682–90. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="plainlinks"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC148477">148477</a> <img alt="Freely accessible" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813" /></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/10373585">10373585</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1093%2Fnar%2F27.13.2682">10.1093/nar/27.13.2682</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ASequence+alignment&amp;rft.atitle=A+comprehensive+comparison+of+multiple+sequence+alignment+programs&amp;rft.au=Thompson+JD%2C+Plewniak+F%2C+Poch+O.&amp;rft.date=1999&amp;rft.genre=article&amp;rft.issue=13&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=2682-90&amp;rft.volume=27&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC148477&amp;rft_id=http%3A%2F%2Fnar.oxfordjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D10373585&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F27.13.2682&amp;rft_id=info%3Apmid%2F10373585&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
</ol>
</div>
<p><a href="/w/index.php?title=Template:Link_FA&amp;action=edit&amp;redlink=1" class="new" title="Template:Link FA (page does not exist)">Template:Link FA</a></p>


<!-- 
NewPP limit report
Parsed by mw1227
Cached time: 20170809204425
Cache expiry: 1900800
Dynamic content: false
CPU time usage: 0.924 seconds
Real time usage: 1.091 seconds
Preprocessor visited node count: 2186/1000000
Preprocessor generated node count: 0/1500000
Post‐expand include size: 98304/2097152 bytes
Template argument size: 1313/2097152 bytes
Highest expansion depth: 12/40
Expensive parser function count: 3/500
Lua time usage: 0.474/10.000 seconds
Lua memory usage: 5.08 MB/50 MB
-->
<!--
Transclusion expansion time report (%,ms,calls,template)
100.00%  855.554      1 -total
 80.10%  685.317      1 Template:Reflist
 48.80%  417.486     35 Template:Cite_journal
 11.03%   94.358      2 Template:Cite_book
  9.97%   85.333      1 Template:Refimprove
  6.54%   55.912      1 Template:Ambox
  2.99%   25.617      2 Template:Dead_link
  2.38%   20.328      2 Template:Fix
  1.79%   15.349      4 Template:Category_handler
  1.76%   15.072      4 Template:Main
-->
</div>