<div class="mw-parser-output"><div class="thumb tright">
<div class="thumbinner" style="width:175px;"><a href="/wiki/File:Cladogram-example1.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/0/08/Cladogram-example1.png" width="173" height="64" class="thumbimage" data-file-width="173" data-file-height="64" /></a>
<div class="thumbcaption">This cladogram shows the relationship among various insect groups. In some cladograms, the length of the horizontal lines indicates time elapsed since the last common ancestor.</div>
</div>
</div>
<p><b>Cladistics</b> (<a href="/wiki/Greek_language" title="Greek language">Greek</a>: <i>clados</i> = branch) or <b>phylogenetic systematics</b> (<a href="/wiki/Greek_language" title="Greek language">Greek</a>: <i>phylon</i> = race and <i>genetic</i> = birth) is a branch of <a href="/wiki/Biology" title="Biology">biology</a> that determines the <a href="/wiki/Evolution" title="Evolution">evolutionary</a> relationships of living things based on derived similarities. It forms the basis for most modern systems of classification, which seek to group organisms by evolutionary relationships. In contrast, <a href="/wiki/Phenetics" title="Phenetics">phenetics</a> groups organisms based on their <i>overall</i> similarity, while more traditional approaches tend to rely on key characters. <a href="/wiki/Willi_Hennig" title="Willi Hennig">Willi Hennig</a> is widely regarded as the founder of cladistics.</p>
<p></p>
<div id="toc" class="toc">
<div class="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#Introduction"><span class="tocnumber">1</span> <span class="toctext">Introduction</span></a></li>
<li class="toclevel-1 tocsection-2"><a href="#Cladistic_methods"><span class="tocnumber">2</span> <span class="toctext">Cladistic methods</span></a></li>
<li class="toclevel-1 tocsection-3"><a href="#Cladistic_classification"><span class="tocnumber">3</span> <span class="toctext">Cladistic classification</span></a></li>
<li class="toclevel-1 tocsection-4"><a href="#See_also"><span class="tocnumber">4</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-5"><a href="#References"><span class="tocnumber">5</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1 tocsection-6"><a href="#External_links"><span class="tocnumber">6</span> <span class="toctext">External links</span></a></li>
</ul>
</div>
<p></p>
<h2><span class="mw-headline" id="Introduction">Introduction</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Cladistics&amp;action=edit&amp;section=1" title="Edit section: Introduction">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Based on a wide variety of information, which includes genetic analysis, biochemical analysis, and analysis of <a href="/wiki/Morphology" class="mw-disambig" title="Morphology">morphology</a>, <a href="/wiki/Tree_(graph_theory)" title="Tree (graph theory)">treelike</a> relationship-diagrams called "cladograms" are drawn up to show different possibilities.</p>
<div class="thumb tright">
<div class="thumbinner" style="width:199px;"><a href="/wiki/File:Cladogram-example2.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/2/2b/Cladogram-example2.png" width="197" height="176" class="thumbimage" data-file-width="197" data-file-height="176" /></a>
<div class="thumbcaption">A vertical orientation yields a cladogram reminiscent of a <a href="/wiki/Tree_(graph_theory)" title="Tree (graph theory)">tree</a>.</div>
</div>
</div>
<p>In a cladogram, all organisms lie at the leaves, and each inner node is ideally binary (two-way). The two <a href="/wiki/Taxon" title="Taxon">taxa</a> on either side of a split are called <i>sister taxa</i> or <i>sister groups</i>. Each subtree, whether it only contains one item or a hundred thousand, is called a <i>clade</i>. A correct cladogram should have all the organisms contained in any one clade share a unique ancestor for that clade, one which they do not share with any other organisms on the diagram. Each clade should be set off by a series of characteristics that appear in its members but not in the other forms it diverged from. These identifying characteristics of a clade are called <a href="/wiki/Synapomorphy" title="Synapomorphy">synapomorphies</a> (shared, derived characters). For instance, hardened front <a href="/wiki/Wing" title="Wing">wings</a> are a synapomorphy of <a href="/wiki/Beetle" title="Beetle">beetles</a>, while <a href="/wiki/Vernation" title="Vernation">circinate vernation</a>, or the unrolling of new fronds, is a synapomorphy of <a href="/wiki/Fern" title="Fern">ferns</a>.</p>
<p>Several more terms are defined for the description of cladograms and the positions of items within them. In an upright cladogram (one with species listed at the top, and a "root" at the bottom), a species or clade is <i>basal</i> to another clade if it branches off toward the bottom, or below the other group in question. Conversely, one clade or species can be described as <i>nested</i> within another. Thus in a cladogram that includes four mammals and a bird, the bird's branch is basal, but a dog would be nested within the mammals. Similarly, a character-state (see below) that seems to be posessed by the common ancestor of a group is described as <i>ancestral</i>, and one that evolved later is comparatively <i>derived</i>. The latter two terms are used instead of "primitive" and "advanced" to avoid placing value-judgements on the evolution of the character states.</p>
<h2><span class="mw-headline" id="Cladistic_methods">Cladistic methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Cladistics&amp;action=edit&amp;section=2" title="Edit section: Cladistic methods">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Typically, an analysis begins by collecting information on certain features of all the organisms in question. Features may come in different versions (i.e. feather-color may be blue in one species, but red in another). These features are collectively called <i>characters,</i> and specific versions are called <i>character states.</i> Thus we might say that "red feathers" and "blue feathers" are two character states of the character "feather-color."</p>
<p>After recording many character states, the researcher then decides which ones were present <i>before</i> the last common ancestor of the group of species (<i>symplesiomorphies</i>) and which were present <i>in</i> the last common ancestor (<i>synapomorphies</i>). Usually this is done by considering some <i>outgroup</i> of organisms we know are not too closely related to any of the organisms in question. Only synapomorphies are of any use in characterising cladistic divisions.</p>
<p>Next, different possible cladograms are drawn up and evaluated. Clades are typically drawn so that they can have as many synapomorphies as possible. The hope is that a sufficiently large number of true syanpomorphies should be large enough to overwhelm any unintended symplesiomorphies (<i>homeoplasies</i>), caused by <a href="/wiki/Convergent_evolution" title="Convergent evolution">convergent evolution</a> (i.e. characters that resemble each other because of environmental conditions or function, and not because of common ancestry. A well-known example of convergent evolution is wings. Though the wings of birds and insects may superficially resemble one another and serve the same function, each evolved independently from the other). If a bird and an insect were both accidentally scored "POSITIVE" for the character "presence of wings", a homeoplasy would be introduced into the dataset, and it might cause erroneous results.</p>
<p>In practice, neutral features like exact <i>ultrastructure</i> (a term for extremely fine structure, microscopic or molecular composition of cellular structure) may be used to provide evidence for real relationships even when the appearance of organisms makes it otherwise difficult. When equivalent possibilities turn up, one is usually chosen based on the principle of <i>parsimony</i>: the most compact arrangement is likely the best (a variation of <a href="/wiki/Occam%27s_razor" title="Occam's razor">Occam's razor</a>). Another approach, particularly useful in molecular evolution, is <a href="/wiki/Maximum_likelihood" class="mw-redirect" title="Maximum likelihood">maximum likelihood</a>, which selects the optimal cladogram that has the highest likelihood based on a specific probability model of changes.</p>
<p>Cladistics has taken a while to settle in, and there is still wide debate over how to apply Hennig's ideas in the real world. In particular, apomorphies are not always easy to distinguish and data are often unavailable due to a sparsity of available forms or a lack of knowledge of characters, and these may invalidate cladograms. There is also concern that use of widely different data sets, for instance structural versus genetic characteristics, may produce widely different trees. However, by and large the phylogenetic approach to systematics has proven useful and coherent and has gained general support.</p>
<p>As <a href="/wiki/DNA" title="DNA">DNA</a> sequencing has become easier, phylogenies are increasingly often constructed with the aid of molecular data. <a href="/wiki/Computational_systematics" class="mw-redirect" title="Computational systematics">Computational systematics</a> allows the use of these large data sets to construct objective phylogenies. These can more accurately filter out true synapomorphy from parallel evolution.</p>
<p>Cladistics does not assume any particular theory of evolution, only the background knowledge of descent with modification. Thus, cladistic methods can be, and recently have been, usefully applied to non-biological systems, including determining language families in <a href="/wiki/Historical_linguistics" title="Historical linguistics">historical linguistics</a> and the <a href="/wiki/Filiation" title="Filiation">filiation</a> of manuscripts in <a href="/wiki/Textual_criticism" title="Textual criticism">textual criticism</a>.</p>
<h2><span class="mw-headline" id="Cladistic_classification">Cladistic classification</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Cladistics&amp;action=edit&amp;section=3" title="Edit section: Cladistic classification">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright">
<div class="thumbinner" style="width:248px;"><a href="/wiki/File:Clade_types.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/0/03/Clade_types.png" width="246" height="218" class="thumbimage" data-file-width="246" data-file-height="218" /></a>
<div class="thumbcaption">Three ways to define a clade for use in a cladistic taxonomy</div>
</div>
</div>
<p>A recent trend in biology since the 1960s, called <b>cladism</b> or <b>cladistic taxonomy</b>, has been to require taxa to be clades. In other words cladists argue that the classification system should be reformed to eliminate all non-clades. In contrast, other evolutionary taxonomists insist that groups reflect <a href="/wiki/Phylogeny" class="mw-redirect" title="Phylogeny">phylogenies</a> and often make use of cladistic techniques, but allow both <a href="/wiki/Monophyletic" class="mw-redirect" title="Monophyletic">monophyletic</a> and <a href="/wiki/Paraphyletic" class="mw-redirect" title="Paraphyletic">paraphyletic</a> groups as <a href="/wiki/Taxa" class="mw-redirect" title="Taxa">taxa</a>.</p>
<p>A <i><a href="/wiki/Monophyletic" class="mw-redirect" title="Monophyletic">monophyletic</a></i> group is a clade, comprising an ancestral form and all of its descendants, and so forming one (and only one) evolutionary group. A <i><a href="/wiki/Paraphyletic" class="mw-redirect" title="Paraphyletic">paraphyletic</a></i> group is similar, but excludes some of the descendants that have undergone significant changes. For instance, the traditional class Reptilia excludes birds even though they evolved from the ancestral reptile. Similarly, the traditional Invertebrates are paraphyletic because Vertebrates are excluded, although the latter evolved from an Invertebrate.</p>
<p>A group with members from separate evolutionary lines is called <i><a href="/wiki/Polyphyletic" class="mw-redirect" title="Polyphyletic">polyphyletic</a></i>. For instance, the once-recognized Pachydermata was found to be polyphyletic because elephants and rhinoceroses arose separately from non-pachyderms. Evolutionary taxonomists consider polyphyletic groups to be errors in classification, often occurring because <a href="/wiki/Convergent_evolution" title="Convergent evolution">convergence</a> or other <a href="/wiki/Homoplasy" class="mw-redirect" title="Homoplasy">homoplasy</a> was misinterpreted as <a href="/wiki/Homology_(biology)" title="Homology (biology)">homology</a>.</p>
<p>Following Hennig, cladists argue that paraphyly are as harmful as polyphyly. The idea is that monophyletic groups can be defined objectively, in terms of common ancestors or the presence of synapomorphies. In contrast, paraphyletic and polyphyletic groups are both defined based on key characters, and the decision of which characters are of taxonomic import is inherently subjective. Many argue that they lead to "gradistic" thinking, where groups advance from "lowly" grades to "advanced" grades, which can in turn lead to the error of <a href="/wiki/Teleology" title="Teleology">teleology</a>.</p>
<p>Going further, some cladists argue that ranks for groups above species are too subjective to present any meaningful information, and so argue that they should be abandoned. Thus they have moved away from Linnaean taxonomy towards a simple hierarchy of clades.</p>
<p>Other evolutionary systematists argue that all taxa are inherently subjective, even when they reflect evolutionary relationships, since living things form an essentially continuous tree. Any dividing line is artificial, and creates both a monophyletic section above and a paraphyletic section below. Paraphyletic taxa are necessary for classifying earlier sections of the tree - for instance, the early vertebrates that would someday evolve into the family Hominidae can not be placed in any other monophyletic family. They also argue that paraphyletic taxa provide information about significant changes in organisms' morphology, ecology, or life history - in short, that both taxa and clades are valuable but distinct notions, with separate purposes. Many use the term <i>monophyly</i> in its older sense, where it includes paraphyly, and use the alternate term <i>holophyly</i> to describe clades (<i>monophyly sensu</i> Hennig).</p>
<p>A formal code of phylogenetic nomenclature, the <a href="/wiki/PhyloCode" title="PhyloCode">PhyloCode</a>, is currently under development for cladistic taxonomy. It is intended for use by both those who would like to abandon Linnaean taxonomy, and by those who would like to use taxa and clades side by side.</p>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Cladistics&amp;action=edit&amp;section=4" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li><a href="/wiki/Scientific_classification" class="mw-redirect" title="Scientific classification">Scientific classification</a></li>
<li><a href="/wiki/Evolutionary_tree" class="mw-redirect" title="Evolutionary tree">tree of life</a></li>
<li><a href="/wiki/Phylogenetic_tree" title="Phylogenetic tree">Phylogenetic tree</a></li>
<li><a href="/wiki/Systematics" title="Systematics">Systematics</a></li>
<li><a href="/wiki/Taxonomy" class="mw-disambig" title="Taxonomy">Taxonomy</a></li>
<li><a href="/wiki/Willi_Hennig" title="Willi Hennig">Willi Hennig</a></li>
<li><a href="/wiki/List_of_publications_in_biology#Cladistics" class="mw-redirect" title="List of publications in biology">Important publications in cladistics</a></li>
</ul>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Cladistics&amp;action=edit&amp;section=5" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li>Kitching IJ, Forey PL, Humphries CJ and Williams DM (1998) Cladistics, 2nd edn. Oxford: Oxford University Press.</li>
<li>Patterson C (1982) Morphological characters and homology. In: Joysey KA and Friday AE (eds) Problems in Phylogenetic Reconstruction. London: Academic Press.</li>
<li>Swofford DL, Olsen GJ, Waddell PJ and Hillis DM (1996) Phylogenetic inference. In: Hillis DM, Moritz C and Mable BK (eds) Molecular Systematics. Sunderland, MA: Sinauer Associates.</li>
<li>de Quieroz K and Gauthier JA (1992) Phylogenetic taxonomy. Annual Review of Ecology and Systematics 23: 449–480.</li>
<li>Wiley EO (1981) Phylogenetics: The Theory and Practice of Phylogenetic Systematics. New York: Wiley Interscience.</li>
</ul>
<h2><span class="mw-headline" id="External_links">External links</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Cladistics&amp;action=edit&amp;section=6" title="Edit section: External links">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li><a rel="nofollow" class="external text" href="http://www.cladistics.org">The Willi Hennig Society</a></li>
<li><a rel="nofollow" class="external text" href="http://www.ucmp.berkeley.edu/clad/clad4.html">Journey into Phylogenetic Systematics</a></li>
<li><a rel="nofollow" class="external text" href="http://tolweb.org/tree/phylogeny.html">Tree of Life Web Project</a></li>
<li><a rel="nofollow" class="external text" href="http://occamssword.com">extensive bibliography</a> for parsimony in Biology and the Philosophy of Biology</li>
<li><a rel="nofollow" class="external text" href="http://rjohara.net/darwin/files/bmcr.html">Example of cladistics used in textual criticism</a></li>
<li><a rel="nofollow" class="external text" href="http://www.amnh.org/learn/pd/fish_2/pdf/compleat_cladist.pdf">The Compleat Cladist (pdf)</a></li>
</ul>


<!-- 
NewPP limit report
Parsed by mw1225
Cached time: 20170809195130
Cache expiry: 1900800
Dynamic content: false
CPU time usage: 0.028 seconds
Real time usage: 0.043 seconds
Preprocessor visited node count: 61/1000000
Preprocessor generated node count: 0/1500000
Post‐expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Highest expansion depth: 2/40
Expensive parser function count: 0/500
-->
<!--
Transclusion expansion time report (%,ms,calls,template)
100.00%    0.000      1 -total
-->
</div>